diff -ur munin-2.0.17/common/lib/Munin/Common/Defaults.pm munin-2.0.17-sep/common/lib/Munin/Common/Defaults.pm
--- munin-2.0.17/common/lib/Munin/Common/Defaults.pm	2013-08-17 12:26:06.386624187 +0200
+++ munin-2.0.17-sep/common/lib/Munin/Common/Defaults.pm	2013-08-17 12:19:22.386149494 +0200
@@ -18,7 +18,8 @@
 our $DROPDOWNLIMIT     = 1;
 
 our $MUNIN_PREFIX     = '';
-our $MUNIN_CONFDIR    = "$COMPONENT_ROOT/t/config/";
+our $MUNIN_MCONFDIR   = "$COMPONENT_ROOT/t/config/";
+our $MUNIN_NCONFDIR   = "$COMPONENT_ROOT/t/config/";
 our $MUNIN_BINDIR     = '';
 our $MUNIN_SBINDIR    = '';
 our $MUNIN_DOCDIR     = '';
Only in munin-2.0.17/common/lib/Munin/Common: Defaults.pm.orig
diff -ur munin-2.0.17/doc/installation/configuration.rst munin-2.0.17-sep/doc/installation/configuration.rst
--- munin-2.0.17/doc/installation/configuration.rst	2013-07-19 18:30:03.000000000 +0200
+++ munin-2.0.17-sep/doc/installation/configuration.rst	2013-08-17 12:23:16.773630125 +0200
@@ -9,7 +9,7 @@
 -------
 
 Decide which plugins to use. The munin node runs all plugins present
-in CONFDIR/plugins/
+in NCONFDIR/plugins/
 
 The quick auto-plug-and-play solution:
 
@@ -59,7 +59,7 @@
 Add some nodes
 --------------
 
-Add some nodes to CONFDIR/munin.conf
+Add some nodes to MCONFDIR/munin.conf
 
 [node.example.com]
   address 192.0.2.4
diff -ur munin-2.0.17/Makefile munin-2.0.17-sep/Makefile
--- munin-2.0.17/Makefile	2013-08-17 12:26:06.386624187 +0200
+++ munin-2.0.17-sep/Makefile	2013-08-17 12:19:22.386149494 +0200
@@ -56,7 +56,7 @@
 # a new config.  Target _only_ suitable for maintainers.
 unconfig:
 	rm -f $(HTMLDIR)/.htaccess
-	rm -f $(CONFDIR)/munin.conf
+	rm -f $(MCONFDIR)/munin.conf
 
 tags:
 	-rm -f TAGS
@@ -71,14 +71,15 @@
 	mkdir -p $(LOGDIR)
 	mkdir -p $(STATEDIR)
 	mkdir -p $(SPOOLDIR)
-	mkdir -p $(CONFDIR)
+	mkdir -p $(NCONFDIR)
+	mkdir -p $(MCONFDIR)
 	$(CHOWN) $(USER) $(LOGDIR) $(STATEDIR) $(SPOOLDIR)
 
 install-master-prime: $(INFILES_MASTER) install-pre install-master
 	mkdir -p $(TEMPLATEDIR)
 	mkdir -p $(TEMPLATEDIR)/static
 	mkdir -p $(TEMPLATEDIR)/partial
-	mkdir -p $(CONFDIR)/munin-conf.d
+	mkdir -p $(MCONFDIR)/munin-conf.d
 	mkdir -p $(LIBDIR)
 	mkdir -p $(BINDIR)
 	mkdir -p $(SBINDIR)
@@ -111,7 +112,7 @@
 	$(INSTALL) -m 0644 master/DejaVuSans.ttf $(LIBDIR)/
 
 	test -f $(HTMLDIR)/.htaccess || $(INSTALL) -m 0644 build/master/www/munin-htaccess $(HTMLDIR)/.htaccess
-	test -f "$(CONFDIR)/munin.conf"  || $(INSTALL) -m 0644 build/master/munin.conf $(CONFDIR)/
+	test -f "$(MCONFDIR)/munin.conf"  || $(INSTALL) -m 0644 build/master/munin.conf $(MCONFDIR)/
 
 	$(INSTALL) -m 0755 build/master/_bin/munin-cron $(SBINDIR)/
 	$(INSTALL) -m 0755 build/master/_bin/munin-check $(LIBDIR)/
@@ -140,14 +141,14 @@
 install-plugins-prime: install-plugins build $(PLUGINS) Makefile Makefile.config
 	@$(CHECKGROUP)
 
-	mkdir -p $(CONFDIR)/plugins
-	mkdir -p $(CONFDIR)/plugin-conf.d
+	mkdir -p $(NCONFDIR)/plugins
+	mkdir -p $(NCONFDIR)/plugin-conf.d
 	mkdir -p $(LIBDIR)/plugins
 	mkdir -p $(PLUGSTATE)
 
 	$(CHOWN) root:root $(PLUGSTATE)
 	$(CHMOD) 0755 $(PLUGSTATE)
-	$(CHMOD) 0755 $(CONFDIR)/plugin-conf.d
+	$(CHMOD) 0755 $(NCONFDIR)/plugin-conf.d
 
 	for p in build/plugins/node.d/* build/plugins/node.d.$(OSTYPE)/* ; do \
 	    if test -f "$$p" ; then                            \
@@ -184,7 +185,7 @@
 install-node-prime: install-node-pre install-node
 
 install-node-pre: build/node/munin-node.conf install-pre
-	test -f "$(CONFDIR)/munin-node.conf" || $(INSTALL) -m 0644 build/node/munin-node.conf $(CONFDIR)/
+	test -f "$(NCONFDIR)/munin-node.conf" || $(INSTALL) -m 0644 build/node/munin-node.conf $(NCONFDIR)/
 
 
 install-common-prime: build-common install-common
@@ -220,7 +221,8 @@
 	@echo "$< -> $@"
 	@mkdir -p build/`dirname $<`
 	@sed -e 's|@@PREFIX@@|$(PREFIX)|g'                      \
-             -e 's|@@CONFDIR@@|$(CONFDIR)|g'                    \
+             -e 's|@@MCONFDIR@@|$(MCONFDIR)|g'                  \
+             -e 's|@@NCONFDIR@@|$(NCONFDIR)|g'                  \
              -e 's|@@BINDIR@@|$(BINDIR)|g'                      \
              -e 's|@@SBINDIR@@|$(SBINDIR)|g'                    \
              -e 's|@@DOCDIR@@|$(DOCDIR)|g'                      \
@@ -261,7 +263,8 @@
 common/blib/lib/Munin/Common/Defaults.pm: common/lib/Munin/Common/Defaults.pm build-common-pre
 	rm -f common/blib/lib/Munin/Common/Defaults.pm
 	$(PERL) -pe 's{(PREFIX     \s+=\s).*}{\1q{$(PREFIX)};}x;   \
-                  s{(CONFDIR    \s+=\s).*}{\1q{$(CONFDIR)};}x;     \
+                  s{(MCONFDIR    \s+=\s).*}{\1q{$(MCONFDIR)};}x;   \
+                  s{(NCONFDIR    \s+=\s).*}{\1q{$(NCONFDIR)};}x;   \
                   s{(BINDIR     \s+=\s).*}{\1q{$(BINDIR)};}x;      \
                   s{(SBINDIR    \s+=\s).*}{\1q{$(SBINDIR)};}x;     \
                   s{(DOCDIR     \s+=\s).*}{\1q{$(DOCDIR)};}x;      \
@@ -388,7 +391,7 @@
 	$(MAKE) $@ CONFIG=t/Makefile.config
 else
 test_plugins = id_default id_root env
-old-test: t/*.t t/install $(addprefix $(CONFDIR)/plugins/,$(test_plugins))
+old-test: t/*.t t/install $(addprefix $(NCONFDIR)/plugins/,$(test_plugins))
 	@for test in t/*.t; do \
 		echo -n "$$test: "; \
 		PERL5LIB=$(PERLLIB) $(PERL) $$test;\
@@ -396,10 +399,10 @@
 endif
 
 node-monkeywrench: install-node
-	rm -rf $(CONFDIR)/plugins
+	rm -rf $(NCONFDIR)/plugins
 	rm -rf $(LIBDIR)/plugins
 	mkdir -p $(LIBDIR)/plugins
-	mkdir -p $(CONFDIR)/plugins
+	mkdir -p $(NCONFDIR)/plugins
 	cp monkeywrench/plugin-break*_ $(LIBDIR)/plugins/
 	$(SBINDIR)/munin-node-configure --suggest
 	echo 'Done?'
diff -ur munin-2.0.17/Makefile.config munin-2.0.17-sep/Makefile.config
--- munin-2.0.17/Makefile.config	2013-08-17 12:26:06.386624187 +0200
+++ munin-2.0.17-sep/Makefile.config	2013-08-17 12:19:22.386149494 +0200
@@ -20,7 +20,8 @@
 PREFIX     = $(DESTDIR)/usr
 
 # Where Munin keeps its configurations (server.conf, client.conf, ++)
-CONFDIR    = $(DESTDIR)/etc/munin
+NCONFDIR    = $(DESTDIR)/etc/munin
+MCONFDIR    = $(DESTDIR)/etc/webapps/munin
 
 # Server only - where to put munin-cron
 BINDIR     = $(PREFIX)/bin
Only in munin-2.0.17: Makefile.config.orig
Only in munin-2.0.17: Makefile.orig
diff -ur munin-2.0.17/master/_bin/munin-cgi-graph.in munin-2.0.17-sep/master/_bin/munin-cgi-graph.in
--- munin-2.0.17/master/_bin/munin-cgi-graph.in	2013-07-19 18:30:03.000000000 +0200
+++ munin-2.0.17-sep/master/_bin/munin-cgi-graph.in	2013-08-17 12:19:22.386149494 +0200
@@ -44,7 +44,7 @@
 use Log::Log4perl qw( :easy );
 
 my $GRAPHER = "$Munin::Common::Defaults::MUNIN_LIBDIR/munin-graph";
-my $conffile = "$Munin::Common::Defaults::MUNIN_CONFDIR/munin.conf";
+my $conffile = "$Munin::Common::Defaults::MUNIN_MCONFDIR/munin.conf";
 
 my %period  = ( "day"   => 300,
 		"week"  => 1800,
diff -ur munin-2.0.17/master/_bin/munin-check.in munin-2.0.17-sep/master/_bin/munin-check.in
--- munin-2.0.17/master/_bin/munin-check.in	2013-07-19 18:30:03.000000000 +0200
+++ munin-2.0.17-sep/master/_bin/munin-check.in	2013-08-17 12:19:22.386149494 +0200
@@ -163,7 +163,7 @@
 norec=yes owner_ok @@PLUGSTATE@@ @@PLUGINUSER@@
 norec=yes perm_ok @@PLUGSTATE@@ 775
 
-norec=yes perm_ok @@CONFDIR@@/plugin-conf.d 755
+norec=yes perm_ok @@NCONFDIR@@/plugin-conf.d 755
 
 echo "Check done.  Please note that this script only checks most things,"
 echo "not all things."
diff -ur munin-2.0.17/master/_bin/munin-graph.in munin-2.0.17-sep/master/_bin/munin-graph.in
--- munin-2.0.17/master/_bin/munin-graph.in	2013-07-19 18:30:03.000000000 +0200
+++ munin-2.0.17-sep/master/_bin/munin-graph.in	2013-08-17 12:19:22.386149494 +0200
@@ -51,7 +51,7 @@
 use Log::Log4perl qw( :easy );
 
 my $GRAPHER = "$Munin::Common::Defaults::MUNIN_LIBDIR/munin-graph";
-my $conffile = "$Munin::Common::Defaults::MUNIN_CONFDIR/munin.conf";
+my $conffile = "$Munin::Common::Defaults::MUNIN_MCONFDIR/munin.conf";
 
 my %period  = ( "day"   => 300,
 		"week"  => 1800,
diff -ur munin-2.0.17/master/_bin/munin-limits.in munin-2.0.17-sep/master/_bin/munin-limits.in
--- munin-2.0.17/master/_bin/munin-limits.in	2013-07-19 18:30:03.000000000 +0200
+++ munin-2.0.17-sep/master/_bin/munin-limits.in	2013-08-17 12:19:22.386149494 +0200
@@ -66,7 +66,7 @@
 
 =item B<< --config <file> >>
 
-Use E<lt>fileE<gt> as configuration file. [@@CONFDIR@@/munin.conf]
+Use E<lt>fileE<gt> as configuration file. [@@MCONFDIR@@/munin.conf]
 
 =item B<< --always-send <severity list> >>
 
@@ -114,7 +114,7 @@
 
 =head1 FILES
 
-	@@CONFDIR@@/munin.conf
+	@@MCONFDIR@@/munin.conf
 	@@DBDIR@@/*
 	@@STATEDIR@@/*
 
diff -ur munin-2.0.17/master/_bin/munin-update.in munin-2.0.17-sep/master/_bin/munin-update.in
--- munin-2.0.17/master/_bin/munin-update.in	2013-07-19 18:30:03.000000000 +0200
+++ munin-2.0.17-sep/master/_bin/munin-update.in	2013-08-17 12:19:22.386149494 +0200
@@ -139,7 +139,7 @@
 
 =item B<< --config_file=<file> >>
 
-Use E<lt>fileE<gt> as the configuration file. [@@CONFDIR@@/munin.conf]
+Use E<lt>fileE<gt> as the configuration file. [@@MCONFDIR@@/munin.conf]
 
 =item B<< --[no]debug >>
 
@@ -192,7 +192,7 @@
 
 =head1 FILES
 
-	@@CONFDIR@@/munin.conf
+	@@MCONFDIR@@/munin.conf
 	@@DBDIR@@/*
 	@@LOGDIR@@/munin-update
 	@@STATEDIR@@/*
diff -ur munin-2.0.17/master/doc/munin.conf.pod.in munin-2.0.17-sep/master/doc/munin.conf.pod.in
--- munin-2.0.17/master/doc/munin.conf.pod.in	2013-08-17 12:26:06.386624187 +0200
+++ munin-2.0.17-sep/master/doc/munin.conf.pod.in	2013-08-17 12:19:22.386149494 +0200
@@ -27,7 +27,7 @@
 	[machine1.your.dom]
 		address localhost
 
-The default location of F<munin.conf> is F<@@CONFDIR@@/munin.conf>.
+The default location of F<munin.conf> is F<@@MCONFDIR@@/munin.conf>.
 If your placement deviates from this norm, use the "--config
 E<lt>fileE<gt>"-option when running the munin-* programs.
 
@@ -149,20 +149,20 @@
 =item B<tls_private_key> <value>
 
 This directive sets the location of the private key to be used for
-TLS.  Default is @@CONFDIR@@/munin.pem.  The private key and
+TLS.  Default is @@MCONFDIR@@/munin.pem.  The private key and
 certificate can be stored in the same file.  Affects: munin-update.
 
 =item B<tls_certificate> <value>
 
 This directive sets the location of the TLS certificate to be used for
-TLS.  Default is @@CONFDIR@@/munin.pem.  The private key and
+TLS.  Default is @@MCONFDIR@@/munin.pem.  The private key and
 certificate can be stored in the same file.  Affects: munin-update.
 
 =item B<tls_ca_certificate> <value>
 
 This directive sets the CA certificate to be used to verify the node's
 certificate, if tls_verify_certificate is set to C<yes>.  Default is
-@@CONFDIR@@/cacert.pem.  Affects: munin-update.
+@@MCONFDIR@@/cacert.pem.  Affects: munin-update.
 
 =item B<tls_verify_depth> <value>
 
Only in munin-2.0.17/master/doc: munin.conf.pod.in.orig
diff -ur munin-2.0.17/master/doc/munin-cron.pod.in munin-2.0.17-sep/master/doc/munin-cron.pod.in
--- munin-2.0.17/master/doc/munin-cron.pod.in	2013-07-19 18:30:03.000000000 +0200
+++ munin-2.0.17-sep/master/doc/munin-cron.pod.in	2013-08-17 12:19:22.386149494 +0200
@@ -22,7 +22,7 @@
 
 =item B<< --config <file> >>
 
-Use E<lt>fileE<gt> as configuration file. [F<@@CONFDIR@@/munin.conf>]
+Use E<lt>fileE<gt> as configuration file. [F<@@MCONFDIR@@/munin.conf>]
 
 =back
 
diff -ur munin-2.0.17/master/extras/munin-update_old.in munin-2.0.17-sep/master/extras/munin-update_old.in
--- munin-2.0.17/master/extras/munin-update_old.in	2013-07-19 18:30:03.000000000 +0200
+++ munin-2.0.17-sep/master/extras/munin-update_old.in	2013-08-17 12:19:22.386149494 +0200
@@ -43,7 +43,7 @@
 my $STATS;
 my $cli_do_fork;
 my $cli_timeout;
-my $conffile = "$Munin::Common::Defaults::MUNIN_CONFDIR/munin.conf";
+my $conffile = "$Munin::Common::Defaults::MUNIN_MCONFDIR/munin.conf";
 my $config;
 my $do_fork = 1;
 my $serversocket  = "munin-server-socket.$$";
@@ -159,7 +159,7 @@
     --host <host>	Limit graphed hosts to <host>. Multiple --host options
     			may be supplied.
     --config <file>	Use <file> as configuration file. 
-    			[$Munin::Common::Defaults::MUNIN_CONFDIR/munin.conf]
+    			[$Munin::Common::Defaults::MUNIN_MCONFDIR/munin.conf]
     --[no]debug		View debug messages. [--nodebug]
     --[no]fork		Don't fork one instance for each host. [--fork]
     --timeout=<seconds>	TCP timeout when talking to clients. [$timeout]
@@ -752,11 +752,11 @@
 	my $ca_cert;
 	my $tls_verify;
         $key = $cert = munin_get ($config, "tls_pem");
-        $key = &munin_get ($config, "tls_private_key", "$Munin::Common::Defaults::MUNIN_CONFDIR/munin.pem")
+        $key = &munin_get ($config, "tls_private_key", "$Munin::Common::Defaults::MUNIN_MCONFDIR/munin.pem")
   	  unless defined $key;
-        $cert = &munin_get ($config, "tls_certificate", "$Munin::Common::Defaults::MUNIN_CONFDIR/munin.pem")
+        $cert = &munin_get ($config, "tls_certificate", "$Munin::Common::Defaults::MUNIN_MCONFDIR/munin.pem")
   	  unless defined $cert;
-        $ca_cert = &munin_get ($config, "tls_ca_certificate", "$Munin::Common::Defaults::MUNIN_CONFDIR/cacert.pem")
+        $ca_cert = &munin_get ($config, "tls_ca_certificate", "$Munin::Common::Defaults::MUNIN_MCONFDIR/cacert.pem")
            unless defined $ca_cert;
         $tls_verify=&munin_get ($config, "tls_verify_certificate", "no");
         $depth=&munin_get ($config, "tls_verify_depth", 5);
@@ -1088,7 +1088,7 @@
 
 =item B<< --config <file> >>
 
-Use E<lt>fileE<gt> as configuration file. [@@CONFDIR@@/munin.conf]
+Use E<lt>fileE<gt> as configuration file. [@@MCONFDIR@@/munin.conf]
 
 =item B<< --help >>
 
@@ -1124,7 +1124,7 @@
 
 =head1 FILES
 
-	@@CONFDIR@@/munin.conf
+	@@MCONFDIR@@/munin.conf
 	@@DBDIR@@/*
 	@@LOGDIR@@/munin-update
 	@@STATEDIR@@/*
diff -ur munin-2.0.17/master/lib/Munin/Master/Config.pm munin-2.0.17-sep/master/lib/Munin/Master/Config.pm
--- munin-2.0.17/master/lib/Munin/Master/Config.pm	2013-08-17 12:26:06.476625774 +0200
+++ munin-2.0.17-sep/master/lib/Munin/Master/Config.pm	2013-08-17 12:19:22.386149494 +0200
@@ -139,7 +139,7 @@
 	    root_instance => 1, 
 
 	    config      => bless ( {
-		config_file      => "$Munin::Common::Defaults::MUNIN_CONFDIR/munin.conf",
+		config_file      => "$Munin::Common::Defaults::MUNIN_MCONFDIR/munin.conf",
 		dbdir            => $Munin::Common::Defaults::MUNIN_DBDIR,
 		debug            => 0,
 		fork             => 1,
@@ -153,9 +153,9 @@
 		rundir           => $Munin::Common::Defaults::MUNIN_STATEDIR,
 		timeout          => 180,
 		tls              => 'disabled',
-		tls_ca_certificate => "$Munin::Common::Defaults::MUNIN_CONFDIR/cacert.pem",
-		tls_certificate  => "$Munin::Common::Defaults::MUNIN_CONFDIR/munin.pem",
-		tls_private_key  => "$Munin::Common::Defaults::MUNIN_CONFDIR/munin.pem",
+		tls_ca_certificate => "$Munin::Common::Defaults::MUNIN_MCONFDIR/cacert.pem",
+		tls_certificate  => "$Munin::Common::Defaults::MUNIN_MCONFDIR/munin.pem",
+		tls_private_key  => "$Munin::Common::Defaults::MUNIN_MCONFDIR/munin.pem",
 		tls_verify_certificate => 0,
 		tls_verify_depth => 5,
 		tmpldir          => "$Munin::Common::Defaults::MUNIN_TEMPLATEDIR",
Only in munin-2.0.17/master/lib/Munin/Master: Config.pm.orig
diff -ur munin-2.0.17/master/lib/Munin/Master/GraphOld.pm munin-2.0.17-sep/master/lib/Munin/Master/GraphOld.pm
--- munin-2.0.17/master/lib/Munin/Master/GraphOld.pm	2013-07-19 18:30:03.000000000 +0200
+++ munin-2.0.17-sep/master/lib/Munin/Master/GraphOld.pm	2013-08-17 12:19:22.386149494 +0200
@@ -90,7 +90,7 @@
 my $skip_stats     = 0;
 my $stdout         = 0;
 my $force_run_as_root = 0;
-my $conffile       = $Munin::Common::Defaults::MUNIN_CONFDIR . "/munin.conf";
+my $conffile       = $Munin::Common::Defaults::MUNIN_MCONFDIR . "/munin.conf";
 my $libdir         = $Munin::Common::Defaults::MUNIN_LIBDIR;
 # Note: Nothing by default is more convenient and elliminates code while
 # for cgi graphing - but it breaks how munin-graph expected stuff to work.
diff -ur munin-2.0.17/master/lib/Munin/Master/HTMLOld.pm munin-2.0.17-sep/master/lib/Munin/Master/HTMLOld.pm
--- munin-2.0.17/master/lib/Munin/Master/HTMLOld.pm	2013-07-19 18:30:03.000000000 +0200
+++ munin-2.0.17-sep/master/lib/Munin/Master/HTMLOld.pm	2013-08-17 12:19:22.389482886 +0200
@@ -81,7 +81,7 @@
 my @times = ("day", "week", "month", "year");
 
 my $DEBUG      = 0;
-my $conffile   = "$Munin::Common::Defaults::MUNIN_CONFDIR/munin.conf";
+my $conffile   = "$Munin::Common::Defaults::MUNIN_MCONFDIR/munin.conf";
 my $do_usage   = 0;
 my $do_version = 0;
 my $stdout     = 0;
@@ -857,7 +857,7 @@
     --service <service>	Compatibility. No effect.
     --host <host>	Compatibility. No effect.
     --config <file>	Use <file> as configuration file. 
-			[/etc/munin/munin.conf]
+			[$Munin::Common::Defaults::MUNIN_MCONFDIR/munin.conf]
 
 ";
     exit 0;
@@ -892,7 +892,7 @@
 
 =item B<< --config <file> >>
 
-Use E<lt>fileE<gt> as configuration file. [/etc/munin/munin.conf]
+Use E<lt>fileE<gt> as configuration file. [@@MCONFDIR@@/munin.conf]
 
 =item B<< --help >>
 
@@ -915,7 +915,7 @@
 
 =head1 FILES
 
-	@@CONFDIR@@/munin.conf
+	@@MCONFDIR@@/munin.conf
 	@@DBDIR@@/datafile
 	@@LOGDIR@@/munin-html
 	@@HTMLDIR@@/*
diff -ur munin-2.0.17/master/lib/Munin/Master/LimitsOld.pm munin-2.0.17-sep/master/lib/Munin/Master/LimitsOld.pm
--- munin-2.0.17/master/lib/Munin/Master/LimitsOld.pm	2013-07-19 18:30:03.000000000 +0200
+++ munin-2.0.17-sep/master/lib/Munin/Master/LimitsOld.pm	2013-08-17 12:19:22.389482886 +0200
@@ -52,7 +52,7 @@
 use Munin::Common::Defaults;
 
 my $DEBUG          = 0;
-my $conffile       = "$Munin::Common::Defaults::MUNIN_CONFDIR/munin.conf";
+my $conffile       = "$Munin::Common::Defaults::MUNIN_MCONFDIR/munin.conf";
 my $do_usage       = 0;
 my $do_version     = 0;
 my @limit_hosts    = ();
@@ -230,7 +230,7 @@
     --contact <contact>	Limit notified contacts to <contact>. Multiple 
     			--contact options may be supplied.
     --config <file>	Use <file> as configuration file. 
-    			[/etc/munin/munin.conf]
+    			[$Munin::Common::Defaults::MUNIN_MCONFDIR/munin.conf]
 
 ";
     exit 0;
Only in munin-2.0.17/master/lib/Munin/Master: Node.pm.orig
diff -ur munin-2.0.17/master/lib/Munin/Master/Utils.pm munin-2.0.17-sep/master/lib/Munin/Master/Utils.pm
--- munin-2.0.17/master/lib/Munin/Master/Utils.pm	2013-08-17 12:26:06.476625774 +0200
+++ munin-2.0.17-sep/master/lib/Munin/Master/Utils.pm	2013-08-17 12:19:22.389482886 +0200
@@ -115,7 +115,7 @@
 	},
 };
 
-my $configfile="$Munin::Common::Defaults::MUNIN_CONFDIR/munin.conf";
+my $configfile="$Munin::Common::Defaults::MUNIN_MCONFDIR/munin.conf";
 
 # Fields to copy when "aliasing" a field
 my @COPY_FIELDS    = ("label", "draw", "type", "rrdfile", "fieldname", "info");
Only in munin-2.0.17/master/lib/Munin/Master: Utils.pm.orig
diff -ur munin-2.0.17/master/munin.conf.in munin-2.0.17-sep/master/munin.conf.in
--- munin-2.0.17/master/munin.conf.in	2013-08-17 12:26:06.476625774 +0200
+++ munin-2.0.17-sep/master/munin.conf.in	2013-08-17 12:19:22.389482886 +0200
@@ -24,7 +24,7 @@
 # cgitmpdir @@DBDIR@@/cgi-tmp
 
 # (Exactly one) directory to include all files from.
-includedir @@CONFDIR@@/munin-conf.d
+includedir @@MCONFDIR@@/munin-conf.d
 
 # You can choose the time reference for "DERIVE" like graphs, and show
 # "per minute", "per hour" values instead of the default "per second"
Only in munin-2.0.17/master: munin.conf.in.orig
diff -ur munin-2.0.17/master/t/munin_master_config.t munin-2.0.17-sep/master/t/munin_master_config.t
--- munin-2.0.17/master/t/munin_master_config.t	2013-07-19 18:30:03.000000000 +0200
+++ munin-2.0.17-sep/master/t/munin_master_config.t	2013-08-17 12:23:31.933897977 +0200
@@ -24,7 +24,7 @@
     },
 
     config => {
-        config_file     => "$Munin::Common::Defaults::MUNIN_CONFDIR/munin.conf",
+        config_file     => "$Munin::Common::Defaults::MUNIN_MCONFDIR/munin.conf",
         dbdir           => '/opt/munin/sandbox/var/opt/munin',
         debug           => 0,
         fork            => 1,
diff -ur munin-2.0.17/master/www/munin-htaccess.in munin-2.0.17-sep/master/www/munin-htaccess.in
--- munin-2.0.17/master/www/munin-htaccess.in	2013-07-19 18:30:03.000000000 +0200
+++ munin-2.0.17-sep/master/www/munin-htaccess.in	2013-08-17 12:19:22.389482886 +0200
@@ -9,7 +9,7 @@
 # 
 # <directory @@HTMLDIR@@>
 
-AuthUserFile @@CONFDIR@@/munin-htpasswd
+AuthUserFile @@MCONFDIR@@/munin-htpasswd
 AuthName "Munin"
 AuthType Basic
 require valid-user
diff -ur munin-2.0.17/node/bin/munindoc munin-2.0.17-sep/node/bin/munindoc
--- munin-2.0.17/node/bin/munindoc	2013-07-19 18:30:03.000000000 +0200
+++ munin-2.0.17-sep/node/bin/munindoc	2013-08-17 12:19:22.389482886 +0200
@@ -48,7 +48,7 @@
 
 # Dirs in which to look for $plugin.pod and $plugin files.
 my @DIRS = ("$Munin::Common::Defaults::MUNIN_LIBDIR/plugins",
-	    "$Munin::Common::Defaults::MUNIN_CONFDIR/plugins",
+	    "$Munin::Common::Defaults::MUNIN_NCONFDIR/plugins",
             "$Munin::Common::Defaults::MUNIN_PERLLIB");
 
 File::Find::find({wanted => \&wanted_pod}, @DIRS);
diff -ur munin-2.0.17/node/doc/munin-node.conf.pod munin-2.0.17-sep/node/doc/munin-node.conf.pod
--- munin-2.0.17/node/doc/munin-node.conf.pod	2013-07-19 18:30:03.000000000 +0200
+++ munin-2.0.17-sep/node/doc/munin-node.conf.pod	2013-08-17 12:24:04.784478220 +0200
@@ -91,20 +91,20 @@
 =item B<tls_private_key> <value>
 
 This directive sets the location of the private key to be used for
-TLS.  Default is @@CONFDIR@@/munin-node.pem.  The private key and
+TLS.  Default is @@NCONFDIR@@/munin-node.pem.  The private key and
 certificate can be stored in the same file.
 
 =item B<tls_certificate> <value>
 
 This directive sets the location of the TLS certificate to be used for
-TLS.  Default is @@CONFDIR@@/munin-node.pem.  The private key and
+TLS.  Default is @@NCONFDIR@@/munin-node.pem.  The private key and
 certificate can be stored in the same file.
 
 =item B<tls_ca_certificate> <value>
 
 This directive sets the CA certificate to be used to verify the node's
 certificate, if tls_verify_certificate is set to C<yes>.  Default is
-@@CONFDIR@@/cacert.pem.
+@@NCONFDIR@@/cacert.pem.
 
 =item B<tls_verify_depth> <value>
 
diff -ur munin-2.0.17/node/extras/munin-node-simple.in munin-2.0.17-sep/node/extras/munin-node-simple.in
--- munin-2.0.17/node/extras/munin-node-simple.in	2013-07-19 18:30:03.000000000 +0200
+++ munin-2.0.17-sep/node/extras/munin-node-simple.in	2013-08-17 12:19:22.389482886 +0200
@@ -28,8 +28,8 @@
 
 $| = 1;
 
-my $clientdir = "@@CONFDIR@@/node.d";
-my $conffile  = "@@CONFDIR@@/node-simple.conf";
+my $clientdir = "@@NCONFDIR@@/node.d";
+my $conffile  = "@@NCONFDIR@@/node-simple.conf";
 my $version   = "@@VERSION@@";
 
 # Empty environment
diff -ur munin-2.0.17/node/lib/Munin/Node/Config.pm munin-2.0.17-sep/node/lib/Munin/Node/Config.pm
--- munin-2.0.17/node/lib/Munin/Node/Config.pm	2013-07-19 18:30:03.000000000 +0200
+++ munin-2.0.17-sep/node/lib/Munin/Node/Config.pm	2013-08-17 12:19:22.389482886 +0200
@@ -25,7 +25,7 @@
         my ($class) = @_;
 
         $instance ||= bless {
-            config_file  => "$Munin::Common::Defaults::MUNIN_CONFDIR/munin-node.conf",
+            config_file  => "$Munin::Common::Defaults::MUNIN_NCONFDIR/munin-node.conf",
         }, $class;
 
         return $instance;
diff -ur munin-2.0.17/node/lib/Munin/Node/Server.pm munin-2.0.17-sep/node/lib/Munin/Node/Server.pm
--- munin-2.0.17/node/lib/Munin/Node/Server.pm	2013-07-19 18:30:03.000000000 +0200
+++ munin-2.0.17-sep/node/lib/Munin/Node/Server.pm	2013-08-17 12:19:22.389482886 +0200
@@ -263,11 +263,11 @@
     my $mode = $session->{tls_mode};
 
     my $key        = $config->{tls_private_key}
-                  || "$Munin::Common::Defaults::MUNIN_CONFDIR/munin-node.pem";
+                  || "$Munin::Common::Defaults::MUNIN_NCONFDIR/munin-node.pem";
     my $cert       = $config->{tls_certificate}
-                  || "$Munin::Common::Defaults::MUNIN_CONFDIR/munin-node.pem";
+                  || "$Munin::Common::Defaults::MUNIN_NCONFDIR/munin-node.pem";
     my $ca_cert    = $config->{tls_ca_certificate}
-                  || "$Munin::Common::Defaults::MUNIN_CONFDIR/cacert.pem";
+                  || "$Munin::Common::Defaults::MUNIN_NCONFDIR/cacert.pem";
     my $tls_verify = $config->{tls_verify_certificate}
                   || 0;
     my $tls_match  = $config->{tls_match};
diff -ur munin-2.0.17/node/lib/Munin/Node/Service.pm munin-2.0.17-sep/node/lib/Munin/Node/Service.pm
--- munin-2.0.17/node/lib/Munin/Node/Service.pm	2013-07-19 18:30:03.000000000 +0200
+++ munin-2.0.17-sep/node/lib/Munin/Node/Service.pm	2013-08-17 12:24:22.264786886 +0200
@@ -22,7 +22,7 @@
     my ($class, %args) = @_;
 
     # Set defaults
-    $args{servicedir} ||= "$Munin::Common::Defaults::MUNIN_CONFDIR/plugins";
+    $args{servicedir} ||= "$Munin::Common::Defaults::MUNIN_NCONFDIR/plugins";
 
     $args{defuser}  ||= getpwnam $Munin::Common::Defaults::MUNIN_PLUGINUSER;
     $args{defgroup} ||= getgrnam $Munin::Common::Defaults::MUNIN_GROUP;
diff -ur munin-2.0.17/node/sbin/munin-node munin-2.0.17-sep/node/sbin/munin-node
--- munin-2.0.17/node/sbin/munin-node	2013-07-19 18:30:03.000000000 +0200
+++ munin-2.0.17-sep/node/sbin/munin-node	2013-08-17 12:19:22.389482886 +0200
@@ -36,8 +36,8 @@
 use Munin::Node::Server;
 
 my $servicedir;
-my $sconfdir = "$Munin::Common::Defaults::MUNIN_CONFDIR/plugin-conf.d";
-my $conffile = "$Munin::Common::Defaults::MUNIN_CONFDIR/munin-node.conf";
+my $sconfdir = "$Munin::Common::Defaults::MUNIN_NCONFDIR/plugin-conf.d";
+my $conffile = "$Munin::Common::Defaults::MUNIN_NCONFDIR/munin-node.conf";
 my $DEBUG    = 0;
 my $PIDEBUG  = 0;
 my $paranoia = 0;
@@ -164,7 +164,7 @@
 
 =item B<< --config <configfile> >>
 
-Use E<lt>fileE<gt> as configuration file. [@@CONFDIR@@/munin-node.conf]
+Use E<lt>fileE<gt> as configuration file. [@@NCONFDIR@@/munin-node.conf]
 
 =item B< --[no]paranoia >
 
@@ -187,9 +187,9 @@
 
 =head1 FILES
 
-    @@CONFDIR@@/munin-node.conf
-    @@CONFDIR@@/plugins/*
-    @@CONFDIR@@/plugin-conf.d/*
+    @@NCONFDIR@@/munin-node.conf
+    @@NCONFDIR@@/plugins/*
+    @@NCONFDIR@@/plugin-conf.d/*
     @@STATEDIR@@/munin-node.pid
     @@LOGDIR@@/munin-node.log
 
diff -ur munin-2.0.17/node/sbin/munin-node-configure munin-2.0.17-sep/node/sbin/munin-node-configure
--- munin-2.0.17/node/sbin/munin-node-configure	2013-07-19 18:30:03.000000000 +0200
+++ munin-2.0.17-sep/node/sbin/munin-node-configure	2013-08-17 12:19:22.489484658 +0200
@@ -76,9 +76,9 @@
 
 sub parse_args
 {
-	my $conffile   = "$Munin::Common::Defaults::MUNIN_CONFDIR/munin-node.conf";
-	my $servicedir = "$Munin::Common::Defaults::MUNIN_CONFDIR/plugins";
-	my $sconfdir   = "$Munin::Common::Defaults::MUNIN_CONFDIR/plugin-conf.d";
+	my $conffile   = "$Munin::Common::Defaults::MUNIN_NCONFDIR/munin-node.conf";
+	my $servicedir = "$Munin::Common::Defaults::MUNIN_NCONFDIR/plugins";
+	my $sconfdir   = "$Munin::Common::Defaults::MUNIN_NCONFDIR/plugin-conf.d";
 	my $libdir     = "$Munin::Common::Defaults::MUNIN_LIBDIR/plugins";
 
 	my $debug = 0;
@@ -492,15 +492,15 @@
 
 =item B<< --config <file> >>
 
-Override configuration file [@@CONFDIR@@/munin-node.conf]
+Override configuration file [@@NCONFDIR@@/munin-node.conf]
 
 =item B<< --servicedir <dir> >>
 
-Override plugin directory [@@CONFDIR@@/plugins/]
+Override plugin directory [@@NCONFDIR@@/plugins/]
 
 =item B<< --sconfdir <dir> >>
 
-Override plugin configuration directory [@@CONFDIR@@/plugin-conf.d/]
+Override plugin configuration directory [@@NCONFDIR@@/plugin-conf.d/]
 
 =item B<< --libdir <dir> >>
 
@@ -648,9 +648,9 @@
 
 =head1 FILES
 
-    @@CONFDIR@@/munin-node.conf
-    @@CONFDIR@@/plugin-conf.d/*
-    @@CONFDIR@@/plugins/*
+    @@NCONFDIR@@/munin-node.conf
+    @@NCONFDIR@@/plugin-conf.d/*
+    @@NCONFDIR@@/plugins/*
     @@LIBDIR@@/plugins/plugins.history
     @@LIBDIR@@/plugins/*
 
diff -ur munin-2.0.17/node/sbin/munin-run munin-2.0.17-sep/node/sbin/munin-run
--- munin-2.0.17/node/sbin/munin-run	2013-07-19 18:30:03.000000000 +0200
+++ munin-2.0.17-sep/node/sbin/munin-run	2013-08-17 12:19:22.489484658 +0200
@@ -35,7 +35,7 @@
 
 my $services;
 my $servicedir;
-my $conffile = "$Munin::Common::Defaults::MUNIN_CONFDIR/munin-node.conf";
+my $conffile = "$Munin::Common::Defaults::MUNIN_NCONFDIR/munin-node.conf";
 my $DEBUG    = 0;
 my $PIDEBUG  = 0;
 my $paranoia = 0;
@@ -98,7 +98,7 @@
 sub parse_args
 {
     # Default configuration values
-    my $sconfdir   = "$Munin::Common::Defaults::MUNIN_CONFDIR/plugin-conf.d";
+    my $sconfdir   = "$Munin::Common::Defaults::MUNIN_NCONFDIR/plugin-conf.d";
     my $sconffile;
 
     my ($plugin, $arg);
@@ -189,15 +189,15 @@
 
 =item B<< --config <configfile> >>
 
-Use E<lt>fileE<gt> as configuration file. [@@CONFDIR@@/munin-node.conf]
+Use E<lt>fileE<gt> as configuration file. [@@NCONFDIR@@/munin-node.conf]
 
 =item B<< --servicedir <dir> >>
 
-Use E<lt>dirE<gt> as plugin dir. [@@CONFDIR@@/plugins/]
+Use E<lt>dirE<gt> as plugin dir. [@@NCONFDIR@@/plugins/]
 
 =item B<< --sconfdir <dir> >>
 
-Use E<lt>dirE<gt> as plugin configuration dir. [@@CONFDIR@@/plugin-conf.d/]
+Use E<lt>dirE<gt> as plugin configuration dir. [@@NCONFDIR@@/plugin-conf.d/]
 
 =item B<< --sconffile <file> >>
 
@@ -230,9 +230,9 @@
 
 =head1 FILES
 
-    @@CONFDIR@@/munin-node.conf
-    @@CONFDIR@@/plugins/*
-    @@CONFDIR@@/plugin-conf.d/*
+    @@NCONFDIR@@/munin-node.conf
+    @@NCONFDIR@@/plugins/*
+    @@NCONFDIR@@/plugin-conf.d/*
     @@STATEDIR@@/munin-node.pid
     @@LOGDIR@@/munin-node.log
 
diff -ur munin-2.0.17/node/sbin/munin-sched munin-2.0.17-sep/node/sbin/munin-sched
--- munin-2.0.17/node/sbin/munin-sched	2013-07-19 18:30:03.000000000 +0200
+++ munin-2.0.17-sep/node/sbin/munin-sched	2013-08-17 12:24:54.562023692 +0200
@@ -34,8 +34,8 @@
 use Munin::Node::ProxySpooler;
 
 my $servicedir;
-my $sconfdir = "$Munin::Common::Defaults::MUNIN_CONFDIR/plugin-conf.d";
-my $conffile = "$Munin::Common::Defaults::MUNIN_CONFDIR/munin-node.conf";
+my $sconfdir = "$Munin::Common::Defaults::MUNIN_NCONFDIR/plugin-conf.d";
+my $conffile = "$Munin::Common::Defaults::MUNIN_NCONFDIR/munin-node.conf";
 my $spooldir = $Munin::Common::Defaults::MUNIN_SPOOLDIR;
 
 my ($host, $port);
@@ -164,7 +164,7 @@
 
 =item B<< --config <configfile> >>
 
-Use E<lt>fileE<gt> as configuration file. [@@CONFDIR@@/munin-node.conf]
+Use E<lt>fileE<gt> as configuration file. [@@NCONFDIR@@/munin-node.conf]
 
 =item B< --[no]paranoia >
 
@@ -182,9 +182,9 @@
 
 =head1 FILES
 
-    @@CONFDIR@@/munin-node.conf
-    @@CONFDIR@@/plugins/*
-    @@CONFDIR@@/plugin-conf.d/*
+    @@NCONFDIR@@/munin-node.conf
+    @@NCONFDIR@@/plugins/*
+    @@NCONFDIR@@/plugin-conf.d/*
     @@STATEDIR@@/munin-sched.pid
     @@LOGDIR@@/munin-sched.log
 
diff -ur munin-2.0.17/plugins/node.d/exim_mailqueue.in munin-2.0.17-sep/plugins/node.d/exim_mailqueue.in
--- munin-2.0.17/plugins/node.d/exim_mailqueue.in	2013-07-19 18:30:03.000000000 +0200
+++ munin-2.0.17-sep/plugins/node.d/exim_mailqueue.in	2013-08-17 12:19:22.489484658 +0200
@@ -15,7 +15,7 @@
   [exim_*]
     user exim
 
-Configuration parameters for a file in @@CONFDIR@@/plugin-conf.d/
+Configuration parameters for a file in @@NCONFDIR@@/plugin-conf.d/
 if you need to override the defaults below:
 
   [exim_mailqueue]
diff -ur munin-2.0.17/plugins/node.d/ipmi_sensor_.in munin-2.0.17-sep/plugins/node.d/ipmi_sensor_.in
--- munin-2.0.17/plugins/node.d/ipmi_sensor_.in	2013-07-19 18:30:03.000000000 +0200
+++ munin-2.0.17-sep/plugins/node.d/ipmi_sensor_.in	2013-08-17 12:19:22.489484658 +0200
@@ -17,7 +17,7 @@
 ipmitool probably needs to be run as root, and it may take more than
 10 seconds on some hosts.
 
-Add the following to your @@CONFDIR@@/munin-node:
+Add the following to your @@NCONFDIR@@/munin-node:
 
  [ipmi_sensor_*]
   user root
@@ -49,7 +49,7 @@
 
 =head1 BUGS
 
-Plugin reads @@CONFDIR@@/ipmi directly, instead of reading environment
+Plugin reads @@NCONFDIR@@/ipmi directly, instead of reading environment
 variables.
 
 =head1 MAGIC MARKERS
@@ -72,7 +72,7 @@
 CACHEDIR = os.environ['MUNIN_PLUGSTATE']
 CACHEFILE = "plugin-ipmi_sensor.cache"
 CACHEAGE = 120
-CONFIG = '@@CONFDIR@@/ipmi'
+CONFIG = '@@NCONFDIR@@/ipmi'
 
 
 
diff -ur munin-2.0.17/plugins/node.d/mysql_innodb.in munin-2.0.17-sep/plugins/node.d/mysql_innodb.in
--- munin-2.0.17/plugins/node.d/mysql_innodb.in	2013-07-19 18:30:03.000000000 +0200
+++ munin-2.0.17-sep/plugins/node.d/mysql_innodb.in	2013-08-17 12:19:22.489484658 +0200
@@ -34,7 +34,7 @@
 
 =head1 CONFIGURATION
 
-Configuration parameters for @@CONFDIR@@/mysql_innodb,
+Configuration parameters for @@NCONFDIR@@/mysql_innodb,
 if you need to override the defaults below:
 
  [mysql_innodb]
diff -ur munin-2.0.17/plugins/node.d/mysql_isam_space_.in munin-2.0.17-sep/plugins/node.d/mysql_isam_space_.in
--- munin-2.0.17/plugins/node.d/mysql_isam_space_.in	2013-07-19 18:30:03.000000000 +0200
+++ munin-2.0.17-sep/plugins/node.d/mysql_isam_space_.in	2013-08-17 12:19:22.489484658 +0200
@@ -8,7 +8,7 @@
 
 =head1 CONFIGURATION
 
-Configuration parameters for @@CONFDIR@@/PLUGIN,
+Configuration parameters for @@NCONFDIR@@/PLUGIN,
 if you need to override the defaults below:
 
  [mysql_isam_space_*]
diff -ur munin-2.0.17/plugins/node.d/mysql_threads.in munin-2.0.17-sep/plugins/node.d/mysql_threads.in
--- munin-2.0.17/plugins/node.d/mysql_threads.in	2013-07-19 18:30:03.000000000 +0200
+++ munin-2.0.17-sep/plugins/node.d/mysql_threads.in	2013-08-17 12:19:22.489484658 +0200
@@ -9,7 +9,7 @@
 
 =head1 CONFIGURATION
 
-Configuration parameters for @@CONFDIR@@/mysql_threads, if you
+Configuration parameters for @@NCONFDIR@@/mysql_threads, if you
 need to override the defaults below:
 
  [mysql_threads]
diff -ur munin-2.0.17/plugins/node.d/named.in munin-2.0.17-sep/plugins/node.d/named.in
--- munin-2.0.17/plugins/node.d/named.in	2013-07-19 18:30:03.000000000 +0200
+++ munin-2.0.17-sep/plugins/node.d/named.in	2013-08-17 12:19:22.489484658 +0200
@@ -34,7 +34,7 @@
 
 =head1 CONFIGURATION
 
-Configuration parameters for @@CONFDIR@@/named,
+Configuration parameters for @@NCONFDIR@@/named,
 if you need to override the defaults below:
 
  [named]
diff -ur munin-2.0.17/plugins/node.d/ntp_kernel_pll_freq.in munin-2.0.17-sep/plugins/node.d/ntp_kernel_pll_freq.in
--- munin-2.0.17/plugins/node.d/ntp_kernel_pll_freq.in	2013-07-19 18:30:03.000000000 +0200
+++ munin-2.0.17-sep/plugins/node.d/ntp_kernel_pll_freq.in	2013-08-17 12:19:22.489484658 +0200
@@ -12,7 +12,7 @@
 
 No configuration
 
-This plugin optionally reads the file @@CONFDIR@@/ntp-freq-comp, which
+This plugin optionally reads the file @@NCONFDIR@@/ntp-freq-comp, which
 should contain a number to be added to the frequency read by ntpdc.
 
 =head1 AUTHORS
@@ -46,8 +46,8 @@
     exit 0
 fi
 
-if [ -f @@CONFDIR@@/ntp-freq-comp ]; then
-    fcomp=`cat @@CONFDIR@@/ntp-freq-comp`
+if [ -f @@NCONFDIR@@/ntp-freq-comp ]; then
+    fcomp=`cat @@NCONFDIR@@/ntp-freq-comp`
 else
     fcomp=0
 fi
diff -ur munin-2.0.17/plugins/node.d/postfix_mailstats.in munin-2.0.17-sep/plugins/node.d/postfix_mailstats.in
--- munin-2.0.17/plugins/node.d/postfix_mailstats.in	2013-07-19 18:30:03.000000000 +0200
+++ munin-2.0.17-sep/plugins/node.d/postfix_mailstats.in	2013-08-17 12:19:22.489484658 +0200
@@ -8,7 +8,7 @@
 
 =head1 CONFIGURATION
 
-Configuration parameters for @@CONFDIR@@/postfix_mailstats,
+Configuration parameters for @@NCONFDIR@@/postfix_mailstats,
 if you need to override the defaults below:
 
  [postfix_mailstats]
diff -ur munin-2.0.17/plugins/node.d/psu_.in munin-2.0.17-sep/plugins/node.d/psu_.in
--- munin-2.0.17/plugins/node.d/psu_.in	2013-07-19 18:30:03.000000000 +0200
+++ munin-2.0.17-sep/plugins/node.d/psu_.in	2013-08-17 12:19:22.489484658 +0200
@@ -15,7 +15,7 @@
 
 Example:
 
- ln -s @@LIBDIR@@/plugins/psu_ @@CONFDIR@@/plugins/psu_foo
+ ln -s @@LIBDIR@@/plugins/psu_ @@NCONFDIR@@/plugins/psu_foo
 
 ...will monitor the user "foo"
 
diff -ur munin-2.0.17/plugins/node.d/squeezebox_.in munin-2.0.17-sep/plugins/node.d/squeezebox_.in
--- munin-2.0.17/plugins/node.d/squeezebox_.in	2013-07-19 18:30:03.000000000 +0200
+++ munin-2.0.17-sep/plugins/node.d/squeezebox_.in	2013-08-17 12:19:22.489484658 +0200
@@ -20,7 +20,7 @@
 non-default configuration, please use the environment variables
 'squeezebox_host' and 'squeezebox_port' to connect. Also, if your
 nc(1) binary is not called "nc", or not the one in $PATH, please define it
-in the plugin's environment file (@@CONFDIR@@/squeezebox or similar).
+in the plugin's environment file (@@NCONFDIR@@/squeezebox or similar).
 
 Sample follows:
 
diff -ur munin-2.0.17/plugins/node.d/varnish_.in munin-2.0.17-sep/plugins/node.d/varnish_.in
--- munin-2.0.17/plugins/node.d/varnish_.in	2013-07-19 18:30:03.000000000 +0200
+++ munin-2.0.17-sep/plugins/node.d/varnish_.in	2013-08-17 12:19:22.492818051 +0200
@@ -54,7 +54,7 @@
 You can link them yourself with something like this:
 
   ln -s @@LIBDIR@@/plugins/varnish_ \
-    @@CONFDIR@@/plugins/varnish_data_structures
+    @@NCONFDIR@@/plugins/varnish_data_structures
 
 =head1 INTERPRETATION
 
diff -ur munin-2.0.17/plugins/node.d.cygwin/df.in munin-2.0.17-sep/plugins/node.d.cygwin/df.in
--- munin-2.0.17/plugins/node.d.cygwin/df.in	2013-07-19 18:30:03.000000000 +0200
+++ munin-2.0.17-sep/plugins/node.d.cygwin/df.in	2013-08-17 12:25:08.005594281 +0200
@@ -13,7 +13,7 @@
 
 =head1 USAGE
 
-Link this plugin to @@CONFDIR@@/plugins/ and restart the munin-node.
+Link this plugin to @@NCONFDIR@@/plugins/ and restart the munin-node.
 
 The plugin excludes per default the following special, read-only or
 dynamically allocating file systems from graphing:
diff -ur munin-2.0.17/plugins/node.d.debug/if.in munin-2.0.17-sep/plugins/node.d.debug/if.in
--- munin-2.0.17/plugins/node.d.debug/if.in	2013-07-19 18:30:03.000000000 +0200
+++ munin-2.0.17-sep/plugins/node.d.debug/if.in	2013-08-17 12:25:17.379093018 +0200
@@ -21,7 +21,7 @@
 
 =head1 USAGE
 
-Link this plugin to @@CONFDIR@@/plugins/ and restart the munin-node.
+Link this plugin to @@NCONFDIR@@/plugins/ and restart the munin-node.
 
 =head1 MAGIC MARKERS
 
diff -ur munin-2.0.17/plugins/node.d.debug/tc.in munin-2.0.17-sep/plugins/node.d.debug/tc.in
--- munin-2.0.17/plugins/node.d.debug/tc.in	2013-07-19 18:30:03.000000000 +0200
+++ munin-2.0.17-sep/plugins/node.d.debug/tc.in	2013-08-17 12:25:26.375918427 +0200
@@ -21,7 +21,7 @@
 
 =head1 USAGE
 
-Link this plugin to @@CONFDIR@@/plugins/ and restart the munin-node.
+Link this plugin to @@NCONFDIR@@/plugins/ and restart the munin-node.
 
 =head1 MAGIC MARKERS
 
diff -ur munin-2.0.17/plugins/node.d.linux/acpi.in munin-2.0.17-sep/plugins/node.d.linux/acpi.in
--- munin-2.0.17/plugins/node.d.linux/acpi.in	2013-07-19 18:30:03.000000000 +0200
+++ munin-2.0.17-sep/plugins/node.d.linux/acpi.in	2013-08-17 12:19:22.492818051 +0200
@@ -17,7 +17,7 @@
 
 =head1 USAGE
 
-Link this plugin to @@CONFDIR@@/plugins/ and restart the munin-node.
+Link this plugin to @@NCONFDIR@@/plugins/ and restart the munin-node.
 
 =head1 INTERPRETATION
 
diff -ur munin-2.0.17/plugins/node.d.linux/apt.in munin-2.0.17-sep/plugins/node.d.linux/apt.in
--- munin-2.0.17/plugins/node.d.linux/apt.in	2013-07-19 18:30:03.000000000 +0200
+++ munin-2.0.17-sep/plugins/node.d.linux/apt.in	2013-08-17 12:19:22.492818051 +0200
@@ -38,7 +38,7 @@
 
 =head1 USAGE
 
-Link this plugin in @@CONFDIR@@/plugins/ and restart the munin-node.
+Link this plugin in @@NCONFDIR@@/plugins/ and restart the munin-node.
 
 =head1 NOTES
 
diff -ur munin-2.0.17/plugins/node.d.linux/df.in munin-2.0.17-sep/plugins/node.d.linux/df.in
--- munin-2.0.17/plugins/node.d.linux/df.in	2013-07-19 18:30:03.000000000 +0200
+++ munin-2.0.17-sep/plugins/node.d.linux/df.in	2013-08-17 12:19:22.492818051 +0200
@@ -28,7 +28,7 @@
     env.warning 92
     env.critical 98
 
-Put it in a file in @@CONFDIR@@/plugin-conf.d/ and restart the munin-node.
+Put it in a file in @@NCONFDIR@@/plugin-conf.d/ and restart the munin-node.
 
 You may specify filesystem specific warning and critical levels:
 
@@ -55,7 +55,7 @@
 
 =head1 USAGE
 
-Link this plugin to @@CONFDIR@@/plugins/ and restart the munin-node.
+Link this plugin to @@NCONFDIR@@/plugins/ and restart the munin-node.
 
 =head1 MAGIC MARKERS
 
diff -ur munin-2.0.17/plugins/node.d.linux/df_inode.in munin-2.0.17-sep/plugins/node.d.linux/df_inode.in
--- munin-2.0.17/plugins/node.d.linux/df_inode.in	2013-07-19 18:30:03.000000000 +0200
+++ munin-2.0.17-sep/plugins/node.d.linux/df_inode.in	2013-08-17 12:25:37.609449943 +0200
@@ -28,7 +28,7 @@
     env.warning 92
     env.critical 98
 
-Put it in a file in @@CONFDIR@@/plugin-conf.d/ and restart the munin-node.
+Put it in a file in @@NCONFDIR@@/plugin-conf.d/ and restart the munin-node.
 
 You may specify filesystem specific warning and critical levels:
 
@@ -55,7 +55,7 @@
 
 =head1 USAGE
 
-Link this plugin to @@CONFDIR@@/plugins/ and restart the munin-node.
+Link this plugin to @@NCONFDIR@@/plugins/ and restart the munin-node.
 
 =head1 MAGIC MARKERS
 
diff -ur munin-2.0.17/plugins/node.d.linux/fw_conntrack.in munin-2.0.17-sep/plugins/node.d.linux/fw_conntrack.in
--- munin-2.0.17/plugins/node.d.linux/fw_conntrack.in	2013-07-19 18:30:03.000000000 +0200
+++ munin-2.0.17-sep/plugins/node.d.linux/fw_conntrack.in	2013-08-17 12:19:22.492818051 +0200
@@ -11,7 +11,7 @@
 
 =head2 CONFIGURATION EXAMPLE
 
-@@CONFDIR@@/plugin-conf.d/global or other file in that dir must contain:
+@@NCONFDIR@@/plugin-conf.d/global or other file in that dir must contain:
 
  [fw_*]
   user root
diff -ur munin-2.0.17/plugins/node.d.linux/fw_forwarded_local.in munin-2.0.17-sep/plugins/node.d.linux/fw_forwarded_local.in
--- munin-2.0.17/plugins/node.d.linux/fw_forwarded_local.in	2013-07-19 18:30:03.000000000 +0200
+++ munin-2.0.17-sep/plugins/node.d.linux/fw_forwarded_local.in	2013-08-17 12:19:22.492818051 +0200
@@ -10,7 +10,7 @@
 
 =head2 CONFIGURATION EXAMPLE
 
-@@CONFDIR@@/plugin-conf.d/global or other file in that dir must contain:
+@@NCONFDIR@@/plugin-conf.d/global or other file in that dir must contain:
 
  [fw_*]
   user root
diff -ur munin-2.0.17/plugins/node.d.linux/fw_packets.in munin-2.0.17-sep/plugins/node.d.linux/fw_packets.in
--- munin-2.0.17/plugins/node.d.linux/fw_packets.in	2013-07-19 18:30:03.000000000 +0200
+++ munin-2.0.17-sep/plugins/node.d.linux/fw_packets.in	2013-08-17 12:19:22.492818051 +0200
@@ -11,7 +11,7 @@
 
 =head1 CONFIGURATION EXAMPLE
 
-@@CONFDIR@@/plugin-conf.d/global or other file in that dir must contain:
+@@NCONFDIR@@/plugin-conf.d/global or other file in that dir must contain:
 
  [fw*]
   user root
diff -ur munin-2.0.17/plugins/node.d.linux/iostat.in munin-2.0.17-sep/plugins/node.d.linux/iostat.in
--- munin-2.0.17/plugins/node.d.linux/iostat.in	2013-07-19 18:30:03.000000000 +0200
+++ munin-2.0.17-sep/plugins/node.d.linux/iostat.in	2013-08-17 12:19:22.492818051 +0200
@@ -24,7 +24,7 @@
 
 =head1 USAGE
 
-Link this plugin to @@CONFDIR@@/plugins/ and restart the munin-node.
+Link this plugin to @@NCONFDIR@@/plugins/ and restart the munin-node.
 
 =head1 INTERPRETATION
 
diff -ur munin-2.0.17/plugins/node.d.linux/iostat_ios.in munin-2.0.17-sep/plugins/node.d.linux/iostat_ios.in
--- munin-2.0.17/plugins/node.d.linux/iostat_ios.in	2013-07-19 18:30:03.000000000 +0200
+++ munin-2.0.17-sep/plugins/node.d.linux/iostat_ios.in	2013-08-17 12:19:22.492818051 +0200
@@ -15,7 +15,7 @@
 
 =head1 USAGE
 
-Link this into @@CONFDIR@@/plugins/ and restart the munin-node.
+Link this into @@NCONFDIR@@/plugins/ and restart the munin-node.
 
 =head1 INTERPRETATION
 
diff -ur munin-2.0.17/plugins/node.d.linux/irqstats.in munin-2.0.17-sep/plugins/node.d.linux/irqstats.in
--- munin-2.0.17/plugins/node.d.linux/irqstats.in	2013-07-19 18:30:03.000000000 +0200
+++ munin-2.0.17-sep/plugins/node.d.linux/irqstats.in	2013-08-17 12:19:22.492818051 +0200
@@ -15,7 +15,7 @@
 
 =head1 USAGE
 
-Link this plugin to @@CONFDIR@@/plugins/ and restart the munin-node.
+Link this plugin to @@NCONFDIR@@/plugins/ and restart the munin-node.
 
 =head1 INTERPRETATION
 
diff -ur munin-2.0.17/resources/hpux-init.d_munin-node.in munin-2.0.17-sep/resources/hpux-init.d_munin-node.in
--- munin-2.0.17/resources/hpux-init.d_munin-node.in	2013-07-19 18:30:03.000000000 +0200
+++ munin-2.0.17-sep/resources/hpux-init.d_munin-node.in	2013-08-17 12:19:22.492818051 +0200
@@ -6,7 +6,7 @@
 PATH=/sbin:/bin:/usr/sbin:/usr/bin
 NAME=munin-node
 DAEMON=@@SBINDIR@@/$NAME
-CONFFILE=@@CONFDIR@@/munin-node.conf
+CONFFILE=@@NCONFDIR@@/munin-node.conf
 RUN_AS=$(awk '$1~/user/{print$2}' $CONFFILE)
 PIDFILE=$(awk '$1~/pid/{print$2}' $CONFFILE)
 MYCONF=/etc/rc.config.d/muninconf
diff -ur munin-2.0.17/resources/linux-cron.d_munin-node.in munin-2.0.17-sep/resources/linux-cron.d_munin-node.in
--- munin-2.0.17/resources/linux-cron.d_munin-node.in	2013-07-19 18:30:03.000000000 +0200
+++ munin-2.0.17-sep/resources/linux-cron.d_munin-node.in	2013-08-17 12:19:22.492818051 +0200
@@ -8,5 +8,5 @@
 # an hour (12 invokations an hour, 1 in 12 chance that the update will
 # happen), but ensure that there will never be more than two hour (7200
 # seconds) interval between updates..
-*/5 * * * *	root [ -x @@CONFDIR@@/plugins/apt ] && @@CONFDIR@@/plugins/apt update 7200 12 >/dev/null
+*/5 * * * *	root [ -x @@NCONFDIR@@/plugins/apt ] && @@NCONFDIR@@/plugins/apt update 7200 12 >/dev/null
 
diff -ur munin-2.0.17/t/Makefile.config munin-2.0.17-sep/t/Makefile.config
--- munin-2.0.17/t/Makefile.config	2013-07-19 18:30:03.000000000 +0200
+++ munin-2.0.17-sep/t/Makefile.config	2013-08-17 12:19:22.492818051 +0200
@@ -9,7 +9,8 @@
 PREFIX     = $(PWD)/t/install
 
 # Where Munin keeps its configurations (server.conf, client.conf, ++)
-CONFDIR    = $(PREFIX)/etc
+MCONFDIR    = $(PREFIX)/etc
+NCONFDIR    = $(PREFIX)/etc
 
 # Client only - Where to put RRD files and other intenal data
 DBDIR      = $(PREFIX)/var/lib
