diff -ur munin-1.4.0/common/lib/Munin/Common/Defaults.pm munin-1.4.0.configs/common/lib/Munin/Common/Defaults.pm
--- munin-1.4.0/common/lib/Munin/Common/Defaults.pm	2009-11-30 17:45:38.228724308 +0100
+++ munin-1.4.0.configs/common/lib/Munin/Common/Defaults.pm	2009-11-30 17:36:32.000000000 +0100
@@ -18,7 +18,8 @@
 our $DROPDOWNLIMIT     = 1;
 
 our $MUNIN_PREFIX     = '';
-our $MUNIN_CONFDIR    = "$COMPONENT_ROOT/t/config/";
+our $MUNIN_MCONFDIR   = "$COMPONENT_ROOT/t/config/";
+our $MUNIN_NCONFDIR   = "$COMPONENT_ROOT/t/config/";
 our $MUNIN_BINDIR     = '';
 our $MUNIN_SBINDIR    = '';
 our $MUNIN_DOCDIR     = '';
diff -ur munin-1.4.0/Makefile munin-1.4.0.configs/Makefile
--- munin-1.4.0/Makefile	2009-11-30 17:45:38.231724115 +0100
+++ munin-1.4.0.configs/Makefile	2009-11-30 17:23:36.000000000 +0100
@@ -50,7 +50,7 @@
 # a new config.  Target _only_ suitable for maintainers.
 unconfig:
 	rm -f $(HTMLDIR)/.htaccess
-	rm -f $(CONFDIR)/munin.conf
+	rm -f $(MCONFDIR)/munin.conf
 
 tags:
 	-rm -f TAGS
@@ -68,13 +68,14 @@
 	@$(CHECKUSER)
 	mkdir -p $(LOGDIR)
 	mkdir -p $(STATEDIR)
-	mkdir -p $(CONFDIR)
+	mkdir -p $(NCONFDIR)
+	mkdir -p $(MCONFDIR)
 	$(CHOWN) $(USER) $(LOGDIR) $(STATEDIR)
 
 install-master-prime: $(INFILES_MASTER) install-pre install-master
 	mkdir -p $(TEMPLATEDIR)
 	mkdir -p $(TEMPLATEDIR)/partial
-	mkdir -p $(CONFDIR)/munin-conf.d
+	mkdir -p $(MCONFDIR)/munin-conf.d
 	mkdir -p $(LIBDIR)
 	mkdir -p $(BINDIR)
 	mkdir -p $(SBINDIR)
@@ -98,7 +99,7 @@
 	$(INSTALL) -m 0755 master/VeraMono.ttf $(LIBDIR)/
 
 	test -f $(HTMLDIR)/.htaccess || $(INSTALL) -m 0644 build/master/www/munin-htaccess $(HTMLDIR)/.htaccess
-	test -f "$(CONFDIR)/munin.conf"  || $(INSTALL) -m 0644 build/master/munin.conf $(CONFDIR)/
+	test -f "$(MCONFDIR)/munin.conf"  || $(INSTALL) -m 0644 build/master/munin.conf $(MCONFDIR)/
 
 	$(INSTALL) -m 0755 build/master/_bin/munin-cron $(SBINDIR)/
 	$(INSTALL) -m 0755 build/master/_bin/munin-check $(LIBDIR)/
@@ -121,14 +122,14 @@
 install-plugins-prime: install-plugins build $(PLUGINS) Makefile Makefile.config
 	@$(CHECKGROUP)
 
-	mkdir -p $(CONFDIR)/plugins
-	mkdir -p $(CONFDIR)/plugin-conf.d
+	mkdir -p $(NCONFDIR)/plugins
+	mkdir -p $(NCONFDIR)/plugin-conf.d
 	mkdir -p $(LIBDIR)/plugins
 	mkdir -p $(PLUGSTATE)
 
 	$(CHOWN) $(PLUGINUSER):$(GROUP) $(PLUGSTATE)
 	$(CHMOD) 0775 $(PLUGSTATE)
-	$(CHMOD) 0755 $(CONFDIR)/plugin-conf.d
+	$(CHMOD) 0755 $(NCONFDIR)/plugin-conf.d
 
 	for p in build/plugins/node.d/* build/plugins/node.d.$(OSTYPE)/* ; do \
 	    if test -f "$$p" ; then                            \
@@ -151,7 +152,7 @@
 install-node-prime: install-node-pre install-node
 
 install-node-pre: build/node/munin-node.conf install-pre
-	test -f "$(CONFDIR)/munin-node.conf" || $(INSTALL) -m 0644 build/node/munin-node.conf $(CONFDIR)/
+	test -f "$(NCONFDIR)/munin-node.conf" || $(INSTALL) -m 0644 build/node/munin-node.conf $(NCONFDIR)/
 
 
 install-common-prime: build-common install-common
@@ -188,7 +189,8 @@
 	@echo "$< -> $@"
 	@mkdir -p build/`dirname $<`
 	@sed -e 's|@@PREFIX@@|$(PREFIX)|g'                      \
-             -e 's|@@CONFDIR@@|$(CONFDIR)|g'                    \
+             -e 's|@@MCONFDIR@@|$(MCONFDIR)|g'                  \
+             -e 's|@@NCONFDIR@@|$(NCONFDIR)|g'                  \
              -e 's|@@BINDIR@@|$(BINDIR)|g'                      \
              -e 's|@@SBINDIR@@|$(SBINDIR)|g'                    \
              -e 's|@@DOCDIR@@|$(DOCDIR)|g'                      \
@@ -229,7 +231,8 @@
 
 common/blib/lib/Munin/Common/Defaults.pm: common/lib/Munin/Common/Defaults.pm build-common-pre
 	$(PERL) -pe 's{(PREFIX     \s+=\s).*}{\1q{$(PREFIX)};}x;   \
-                  s{(CONFDIR    \s+=\s).*}{\1q{$(CONFDIR)};}x;     \
+                  s{(MCONFDIR    \s+=\s).*}{\1q{$(MCONFDIR)};}x;   \
+                  s{(NCONFDIR    \s+=\s).*}{\1q{$(NCONFDIR)};}x;   \
                   s{(BINDIR     \s+=\s).*}{\1q{$(BINDIR)};}x;      \
                   s{(SBINDIR    \s+=\s).*}{\1q{$(SBINDIR)};}x;     \
                   s{(DOCDIR     \s+=\s).*}{\1q{$(DOCDIR)};}x;      \
@@ -373,7 +376,7 @@
 	$(MAKE) $@ CONFIG=t/Makefile.config
 else
 test_plugins = id_default id_root env
-old-test: t/*.t t/install $(addprefix $(CONFDIR)/plugins/,$(test_plugins))
+old-test: t/*.t t/install $(addprefix $(NCONFDIR)/plugins/,$(test_plugins))
 	@for test in t/*.t; do \
 		echo -n "$$test: "; \
 		PERL5LIB=$(PERLLIB) $(PERL) $$test;\
@@ -381,10 +384,10 @@
 endif
 
 node-monkeywrench: install-node
-	rm -rf $(CONFDIR)/plugins
+	rm -rf $(NCONFDIR)/plugins
 	rm -rf $(LIBDIR)/plugins
 	mkdir -p $(LIBDIR)/plugins
-	mkdir -p $(CONFDIR)/plugins
+	mkdir -p $(NCONFDIR)/plugins
 	cp monkeywrench/plugin-break*_ $(LIBDIR)/plugins/
 	$(SBINDIR)/munin-node-configure --suggest
 	echo 'Done?'
diff -ur munin-1.4.0/Makefile.config munin-1.4.0.configs/Makefile.config
--- munin-1.4.0/Makefile.config	2009-11-30 17:45:38.234724358 +0100
+++ munin-1.4.0.configs/Makefile.config	2009-11-30 17:22:04.000000000 +0100
@@ -20,7 +20,8 @@
 PREFIX     = $(DESTDIR)/usr
 
 # Where Munin keeps its configurations (server.conf, client.conf, ++)
-CONFDIR    = $(DESTDIR)/etc/munin
+NCONFDIR    = $(DESTDIR)/etc/munin
+MCONFDIR    = $(DESTDIR)/etc/webapps/munin
 
 # Server only - where to put munin-cron
 BINDIR     = $(PREFIX)/bin
diff -ur munin-1.4.0/master/_bin/munin-cgi-graph.in munin-1.4.0.configs/master/_bin/munin-cgi-graph.in
--- munin-1.4.0/master/_bin/munin-cgi-graph.in	2009-11-27 12:17:18.000000000 +0100
+++ munin-1.4.0.configs/master/_bin/munin-cgi-graph.in	2009-11-30 17:24:36.000000000 +0100
@@ -32,7 +32,7 @@
 use IPC::SysV qw(IPC_CREAT);
 
 my $GRAPHER = "@@LIBDIR@@/munin-graph";
-my $conffile = "@@CONFDIR@@/munin.conf";
+my $conffile = "@@MCONFDIR@@/munin.conf";
 
 my %TIMES   = ( "day"   => ["--noweek", "--nomonth", "--noyear", "--nosumweek", "--nosumyear"], 
 		"week"  => ["--noday", "--nomonth", "--noyear", "--nosumweek", "--nosumyear"], 
diff -ur munin-1.4.0/master/_bin/munin-check.in munin-1.4.0.configs/master/_bin/munin-check.in
--- munin-1.4.0/master/_bin/munin-check.in	2009-11-27 12:17:18.000000000 +0100
+++ munin-1.4.0.configs/master/_bin/munin-check.in	2009-11-30 17:24:54.000000000 +0100
@@ -166,7 +166,7 @@
 norec=yes owner_ok @@PLUGSTATE@@ @@PLUGINUSER@@
 norec=yes perm_ok @@PLUGSTATE@@ 775
 
-norec=yes perm_ok @@CONFDIR@@/plugin-conf.d 755
+norec=yes perm_ok @@NCONFDIR@@/plugin-conf.d 755
 
 echo "Check done.  Please note that this script only checks most things,"
 echo "not all things."
diff -ur munin-1.4.0/master/_bin/munin-fastcgi-graph.in munin-1.4.0.configs/master/_bin/munin-fastcgi-graph.in
--- munin-1.4.0/master/_bin/munin-fastcgi-graph.in	2009-11-27 12:17:18.000000000 +0100
+++ munin-1.4.0.configs/master/_bin/munin-fastcgi-graph.in	2009-11-30 17:25:36.000000000 +0100
@@ -32,7 +32,7 @@
 use CGI::Fast;
 
 my $GRAPHER = "@@LIBDIR@@/munin-graph";
-my $conffile = "@@CONFDIR@@/munin.conf";
+my $conffile = "@@MCONFDIR@@/munin.conf";
 
 my %TIMES   = ( "day"   => ["--noweek", "--nomonth", "--noyear", "--nosumweek", "--nosumyear"], 
 		"week"  => ["--noday", "--nomonth", "--noyear", "--nosumweek", "--nosumyear"], 
diff -ur munin-1.4.0/master/_bin/munin-gather.in munin-1.4.0.configs/master/_bin/munin-gather.in
--- munin-1.4.0/master/_bin/munin-gather.in	2009-11-27 12:17:18.000000000 +0100
+++ munin-1.4.0.configs/master/_bin/munin-gather.in	2009-11-30 17:26:10.000000000 +0100
@@ -40,7 +40,7 @@
 my $MDEBUG          = 0;
 my $VERSION        = "@@VERSION@@";
 my $serversocket   = "munin-server-socket.$$";
-my $conffile       = "@@CONFDIR@@/munin.conf";
+my $conffile       = "@@MCONFDIR@@/munin.conf";
 my $force_root     = 0;
 my $do_usage       = 0;
 my @limit_hosts    = ();
@@ -84,7 +84,7 @@
     --host <host>	Limit graphed hosts to <host>. Multiple --host options
     			may be supplied.
     --config <file>	Use <file> as configuration file.
-    			[@@CONFDIR@@/munin.conf]
+    			[@@MCONFDIR@@/munin.conf]
     --log <level>       Set log level: DEBUG|INFO|WARN|ERROR|FATAL.
                         These levels work the same way as in syslog.
     --[no]debug		Send debug messages both to the log and to the screen
@@ -827,13 +827,13 @@
         my $tls_verify;
 
         $key = $cert = munin_get( $config, "tls_pem" );
-        $key = &munin_get( $config, "tls_private_key", "@@CONFDIR@@/munin.pem" )
+        $key = &munin_get( $config, "tls_private_key", "@@MCONFDIR@@/munin.pem" )
           unless defined $key;
         $cert =
-          &munin_get( $config, "tls_certificate", "@@CONFDIR@@/munin.pem" )
+          &munin_get( $config, "tls_certificate", "@@MCONFDIR@@/munin.pem" )
           unless defined $cert;
         $ca_cert =
-          &munin_get( $config, "tls_ca_certificate", "@@CONFDIR@@/cacert.pem" )
+          &munin_get( $config, "tls_ca_certificate", "@@MCONFDIR@@/cacert.pem" )
           unless defined $ca_cert;
         $tls_verify = &munin_get( $config, "tls_verify_certificate", "no" );
         $depth      = &munin_get( $config, "tls_verify_depth",       5 );
@@ -919,7 +919,7 @@
 
 =item B<< --config <file> >>
 
-Use E<lt>fileE<gt> as configuration file. [@@CONFDIR@@/munin.conf]
+Use E<lt>fileE<gt> as configuration file. [@@MCONFDIR@@/munin.conf]
 
 =item B<< --help >>
 
@@ -959,7 +959,7 @@
 
 =head1 FILES
 
-	@@CONFDIR@@/munin.conf
+	@@MCONFDIR@@/munin.conf
 	@@DBDIR@@/*
 	@@LOGDIR@@/munin-update
 	@@STATEDIR@@/*
diff -ur munin-1.4.0/master/_bin/munin-graph.in munin-1.4.0.configs/master/_bin/munin-graph.in
--- munin-1.4.0/master/_bin/munin-graph.in	2009-11-27 12:17:18.000000000 +0100
+++ munin-1.4.0.configs/master/_bin/munin-graph.in	2009-11-30 17:26:27.000000000 +0100
@@ -92,7 +92,7 @@
 
 =item B<< --config <file> >>
 
-Use E<lt>fileE<gt> as configuration file. [@@CONFDIR@@/munin.conf]
+Use E<lt>fileE<gt> as configuration file. [@@MCONFDIR@@/munin.conf]
 
 =item B<< --[no]list-images >>
 
@@ -135,7 +135,7 @@
 
 =head1 FILES
 
-	@@CONFDIR@@/munin.conf
+	@@MCONFDIR@@/munin.conf
 	@@DBDIR@@/*
 	@@LOGDIR@@/munin-graph
 	@@STATEDIR@@/*
diff -ur munin-1.4.0/master/_bin/munin-html.in munin-1.4.0.configs/master/_bin/munin-html.in
--- munin-1.4.0/master/_bin/munin-html.in	2009-11-27 12:17:18.000000000 +0100
+++ munin-1.4.0.configs/master/_bin/munin-html.in	2009-11-30 17:26:34.000000000 +0100
@@ -90,7 +90,7 @@
 
 =head1 FILES
 
-	@@CONFDIR@@/munin.conf
+	@@MCONFDIR@@/munin.conf
 	@@DBDIR@@/datafile
 	@@LOGDIR@@/munin-html
 	@@HTMLDIR@@/*
diff -ur munin-1.4.0/master/_bin/munin-limits.in munin-1.4.0.configs/master/_bin/munin-limits.in
--- munin-1.4.0/master/_bin/munin-limits.in	2009-11-27 12:17:18.000000000 +0100
+++ munin-1.4.0.configs/master/_bin/munin-limits.in	2009-11-30 17:26:41.000000000 +0100
@@ -64,7 +64,7 @@
 
 =item B<< --config <file> >>
 
-Use E<lt>fileE<gt> as configuration file. [@@CONFDIR@@/munin.conf]
+Use E<lt>fileE<gt> as configuration file. [@@MCONFDIR@@/munin.conf]
 
 =item B<< --[no]force >>
 
@@ -100,7 +100,7 @@
 
 =head1 FILES
 
-	@@CONFDIR@@/munin.conf
+	@@MCONFDIR@@/munin.conf
 	@@DBDIR@@/*
 	@@STATEDIR@@/*
 
diff -ur munin-1.4.0/master/_bin/munin-update.in munin-1.4.0.configs/master/_bin/munin-update.in
--- munin-1.4.0/master/_bin/munin-update.in	2009-11-27 12:17:18.000000000 +0100
+++ munin-1.4.0.configs/master/_bin/munin-update.in	2009-11-30 17:26:56.000000000 +0100
@@ -137,7 +137,7 @@
 
 =item B<< --config_file=<file> >>
 
-Use E<lt>fileE<gt> as the configuration file. [@@CONFDIR@@/munin.conf]
+Use E<lt>fileE<gt> as the configuration file. [@@MCONFDIR@@/munin.conf]
 
 =item B<< --[no]debug >>
 
@@ -190,7 +190,7 @@
 
 =head1 FILES
 
-	@@CONFDIR@@/munin.conf
+	@@MCONFDIR@@/munin.conf
 	@@DBDIR@@/*
 	@@LOGDIR@@/munin-update
 	@@STATEDIR@@/*
diff -ur munin-1.4.0/master/doc/munin.conf.pod.in munin-1.4.0.configs/master/doc/munin.conf.pod.in
--- munin-1.4.0/master/doc/munin.conf.pod.in	2009-11-30 17:45:38.237723844 +0100
+++ munin-1.4.0.configs/master/doc/munin.conf.pod.in	2009-11-30 17:31:11.000000000 +0100
@@ -25,7 +25,7 @@
 	[machine1.your.dom]
 		address localhost
 
-The default location of F<munin.conf> is F<@@CONFDIR@@/munin.conf>.
+The default location of F<munin.conf> is F<@@MCONFDIR@@/munin.conf>.
 If your placement deviates from this norm, use the "--config
 E<lt>fileE<gt>"-option when running the munin-* programs.
 
@@ -153,20 +153,20 @@
 =item B<tls_private_key> <value>
 
 This directive sets the location of the private key to be used for
-TLS.  Default is @@CONFDIR@@/munin.pem.  The private key and
+TLS.  Default is @@MCONFDIR@@/munin.pem.  The private key and
 certificate can be stored in the same file.  Affects: munin-update.
 
 =item B<tls_certificate> <value>
 
 This directive sets the location of the TLS certificate to be used for
-TLS.  Default is @@CONFDIR@@/munin.pem.  The private key and
+TLS.  Default is @@MCONFDIR@@/munin.pem.  The private key and
 certificate can be stored in the same file.  Affects: munin-update.
 
 =item B<tls_ca_certificate> <value>
 
 This directive sets the CA certificate to be used to verify the node's
 certificate, if tls_verify_certificate is set to C<yes>.  Default is
-@@CONFDIR@@/cacert.pem.  Affects: munin-update.
+@@MCONFDIR@@/cacert.pem.  Affects: munin-update.
 
 =item B<tls_verify_depth> <value>
 
diff -ur munin-1.4.0/master/doc/munin-cron.pod.in munin-1.4.0.configs/master/doc/munin-cron.pod.in
--- munin-1.4.0/master/doc/munin-cron.pod.in	2009-11-27 12:17:16.000000000 +0100
+++ munin-1.4.0.configs/master/doc/munin-cron.pod.in	2009-11-30 17:31:17.000000000 +0100
@@ -20,7 +20,7 @@
 
 =item B<< --config <file> >>
 
-Use E<lt>fileE<gt> as configuration file. [F<@@CONFDIR@@/munin.conf>]
+Use E<lt>fileE<gt> as configuration file. [F<@@MCONFDIR@@/munin.conf>]
 
 =back
 
diff -ur munin-1.4.0/master/extras/munin-update_old.in munin-1.4.0.configs/master/extras/munin-update_old.in
--- munin-1.4.0/master/extras/munin-update_old.in	2009-11-27 12:17:17.000000000 +0100
+++ munin-1.4.0.configs/master/extras/munin-update_old.in	2009-11-30 17:31:58.000000000 +0100
@@ -43,7 +43,7 @@
 my $STATS;
 my $cli_do_fork;
 my $cli_timeout;
-my $conffile = "$Munin::Common::Defaults::MUNIN_CONFDIR/munin.conf";
+my $conffile = "$Munin::Common::Defaults::MUNIN_MCONFDIR/munin.conf";
 my $config;
 my $do_fork = 1;
 my $serversocket  = "munin-server-socket.$$";
@@ -161,7 +161,7 @@
     --host <host>	Limit graphed hosts to <host>. Multiple --host options
     			may be supplied.
     --config <file>	Use <file> as configuration file. 
-    			[$Munin::Common::Defaults::MUNIN_CONFDIR/munin.conf]
+    			[$Munin::Common::Defaults::MUNIN_MCONFDIR/munin.conf]
     --[no]debug		View debug messages. [--nodebug]
     --[no]fork		Don't fork one instance for each host. [--fork]
     --timeout=<seconds>	TCP timeout when talking to clients. [$timeout]
@@ -754,11 +754,11 @@
 	my $ca_cert;
 	my $tls_verify;
         $key = $cert = munin_get ($config, "tls_pem");
-        $key = &munin_get ($config, "tls_private_key", "$Munin::Common::Defaults::MUNIN_CONFDIR/munin.pem")
+        $key = &munin_get ($config, "tls_private_key", "$Munin::Common::Defaults::MUNIN_MCONFDIR/munin.pem")
   	  unless defined $key;
-        $cert = &munin_get ($config, "tls_certificate", "$Munin::Common::Defaults::MUNIN_CONFDIR/munin.pem")
+        $cert = &munin_get ($config, "tls_certificate", "$Munin::Common::Defaults::MUNIN_MCONFDIR/munin.pem")
   	  unless defined $cert;
-        $ca_cert = &munin_get ($config, "tls_ca_certificate", "$Munin::Common::Defaults::MUNIN_CONFDIR/cacert.pem")
+        $ca_cert = &munin_get ($config, "tls_ca_certificate", "$Munin::Common::Defaults::MUNIN_MCONFDIR/cacert.pem")
            unless defined $ca_cert;
         $tls_verify=&munin_get ($config, "tls_verify_certificate", "no");
         $depth=&munin_get ($config, "tls_verify_depth", 5);
@@ -1090,7 +1090,7 @@
 
 =item B<< --config <file> >>
 
-Use E<lt>fileE<gt> as configuration file. [@@CONFDIR@@/munin.conf]
+Use E<lt>fileE<gt> as configuration file. [@@MCONFDIR@@/munin.conf]
 
 =item B<< --help >>
 
@@ -1126,7 +1126,7 @@
 
 =head1 FILES
 
-	@@CONFDIR@@/munin.conf
+	@@MCONFDIR@@/munin.conf
 	@@DBDIR@@/*
 	@@LOGDIR@@/munin-update
 	@@STATEDIR@@/*
diff -ur munin-1.4.0/master/lib/Munin/Master/Config.pm munin-1.4.0.configs/master/lib/Munin/Master/Config.pm
--- munin-1.4.0/master/lib/Munin/Master/Config.pm	2009-11-30 17:45:38.241724215 +0100
+++ munin-1.4.0.configs/master/lib/Munin/Master/Config.pm	2009-11-30 17:28:11.000000000 +0100
@@ -139,7 +139,7 @@
 	    root_instance => 1, 
 
 	    config      => bless ( {
-		config_file      => "$Munin::Common::Defaults::MUNIN_CONFDIR/munin.conf",
+		config_file      => "$Munin::Common::Defaults::MUNIN_MCONFDIR/munin.conf",
 		dbdir            => $Munin::Common::Defaults::MUNIN_DBDIR,
 		debug            => 0,
 		fork             => 1,
@@ -151,9 +151,9 @@
 		rundir           => '/tmp',
 		timeout          => 180,
 		tls              => 'disabled',
-		tls_ca_certificate => "$Munin::Common::Defaults::MUNIN_CONFDIR/cacert.pem",
-		tls_certificate  => "$Munin::Common::Defaults::MUNIN_CONFDIR/munin.pem",
-		tls_private_key  => "$Munin::Common::Defaults::MUNIN_CONFDIR/munin.pem",
+		tls_ca_certificate => "$Munin::Common::Defaults::MUNIN_MCONFDIR/cacert.pem",
+		tls_certificate  => "$Munin::Common::Defaults::MUNIN_MCONFDIR/munin.pem",
+		tls_private_key  => "$Munin::Common::Defaults::MUNIN_MCONFDIR/munin.pem",
 		tls_verify_certificate => 0,
 		tls_verify_depth => 5,
 		tmpldir          => $Munin::Common::Defaults::MUNIN_TEMPLATEDIR,
diff -ur munin-1.4.0/master/lib/Munin/Master/GraphOld.pm munin-1.4.0.configs/master/lib/Munin/Master/GraphOld.pm
--- munin-1.4.0/master/lib/Munin/Master/GraphOld.pm	2009-11-27 12:17:17.000000000 +0100
+++ munin-1.4.0.configs/master/lib/Munin/Master/GraphOld.pm	2009-11-30 17:28:20.000000000 +0100
@@ -89,7 +89,7 @@
 my $skip_locking   = 0;
 my $skip_stats     = 0;
 my $stdout         = 0;
-my $conffile       = $Munin::Common::Defaults::MUNIN_CONFDIR . "/munin.conf";
+my $conffile       = $Munin::Common::Defaults::MUNIN_MCONFDIR . "/munin.conf";
 my $libdir         = $Munin::Common::Defaults::MUNIN_LIBDIR;
 my %draw           = (
     "day"     => 1,
diff -ur munin-1.4.0/master/lib/Munin/Master/HTMLOld.pm munin-1.4.0.configs/master/lib/Munin/Master/HTMLOld.pm
--- munin-1.4.0/master/lib/Munin/Master/HTMLOld.pm	2009-11-27 12:17:18.000000000 +0100
+++ munin-1.4.0.configs/master/lib/Munin/Master/HTMLOld.pm	2009-11-30 17:30:13.000000000 +0100
@@ -77,7 +77,7 @@
 my @times = ("day", "week", "month", "year");
 
 my $DEBUG      = 0;
-my $conffile   = "$Munin::Common::Defaults::MUNIN_CONFDIR/munin.conf";
+my $conffile   = "$Munin::Common::Defaults::MUNIN_MCONFDIR/munin.conf";
 my $do_usage   = 0;
 my $do_version = 0;
 my $stdout     = 0;
@@ -1068,7 +1068,7 @@
     --service <service>	Compatability. No effect.
     --host <host>	Compatability. No effect.
     --config <file>	Use <file> as configuration file. 
-			[/etc/munin/munin.conf]
+			[@@MCONFDIR@@/munin.conf]
 
 ";
     exit 0;
@@ -1103,7 +1103,7 @@
 
 =item B<< --config <file> >>
 
-Use E<lt>fileE<gt> as configuration file. [/etc/munin/munin.conf]
+Use E<lt>fileE<gt> as configuration file. [@@MCONFDIR@@/munin.conf]
 
 =item B<< --help >>
 
@@ -1130,7 +1130,7 @@
 
 =head1 FILES
 
-	@@CONFDIR@@/munin.conf
+	@@MCONFDIR@@/munin.conf
 	@@DBDIR@@/datafile
 	@@LOGDIR@@/munin-html
 	@@HTMLDIR@@/*
diff -ur munin-1.4.0/master/lib/Munin/Master/LimitsOld.pm munin-1.4.0.configs/master/lib/Munin/Master/LimitsOld.pm
--- munin-1.4.0/master/lib/Munin/Master/LimitsOld.pm	2009-11-27 12:17:18.000000000 +0100
+++ munin-1.4.0.configs/master/lib/Munin/Master/LimitsOld.pm	2009-11-30 17:29:45.000000000 +0100
@@ -53,7 +53,7 @@
 use Munin::Common::Defaults;
 
 my $DEBUG          = 0;
-my $conffile       = "$Munin::Common::Defaults::MUNIN_CONFDIR/munin.conf";
+my $conffile       = "$Munin::Common::Defaults::MUNIN_MCONFDIR/munin.conf";
 my $do_usage       = 0;
 my $do_version     = 0;
 my @limit_hosts    = ();
@@ -199,7 +199,7 @@
     --contact <contact>	Limit notified contacts to <contact>. Multiple 
     			--contact options may be supplied.
     --config <file>	Use <file> as configuration file. 
-    			[/etc/munin/munin.conf]
+    			[@@MCONFDIR@@/munin.conf]
 
 ";
     exit 0;
diff -ur munin-1.4.0/master/lib/Munin/Master/Utils.pm munin-1.4.0.configs/master/lib/Munin/Master/Utils.pm
--- munin-1.4.0/master/lib/Munin/Master/Utils.pm	2009-11-30 17:45:38.244724521 +0100
+++ munin-1.4.0.configs/master/lib/Munin/Master/Utils.pm	2009-11-30 17:29:03.000000000 +0100
@@ -78,7 +78,7 @@
 my $nsca = new IO::Handle;
 my $config = undef;
 
-my $configfile="$Munin::Common::Defaults::MUNIN_CONFDIR/munin.conf";
+my $configfile="$Munin::Common::Defaults::MUNIN_MCONFDIR/munin.conf";
 
 # Fields to copy when "aliasing" a field
 my @COPY_FIELDS    = ("label", "draw", "type", "rrdfile", "fieldname", "info");
diff -ur munin-1.4.0/master/munin.conf.in munin-1.4.0.configs/master/munin.conf.in
--- munin-1.4.0/master/munin.conf.in	2009-11-30 17:45:38.246730214 +0100
+++ munin-1.4.0.configs/master/munin.conf.in	2009-11-30 17:38:23.000000000 +0100
@@ -15,7 +15,7 @@
 
 # (Exactly one) directory to include all files from.
 #
-includedir @@CONFDIR@@/munin-conf.d
+includedir @@MCONFDIR@@/munin-conf.d
 
 # Make graphs show values per minute instead of per second
 #graph_period minute
diff -ur munin-1.4.0/master/www/munin-htaccess.in munin-1.4.0.configs/master/www/munin-htaccess.in
--- munin-1.4.0/master/www/munin-htaccess.in	2009-11-27 12:17:16.000000000 +0100
+++ munin-1.4.0.configs/master/www/munin-htaccess.in	2009-11-30 17:37:25.000000000 +0100
@@ -9,7 +9,7 @@
 # 
 # <directory @@HTMLDIR@@>
 
-AuthUserFile @@CONFDIR@@/munin-htpasswd
+AuthUserFile @@MCONFDIR@@/munin-htpasswd
 AuthName "Munin"
 AuthType Basic
 require valid-user
diff -ur munin-1.4.0/node/bin/munindoc munin-1.4.0.configs/node/bin/munindoc
--- munin-1.4.0/node/bin/munindoc	2009-11-27 12:17:13.000000000 +0100
+++ munin-1.4.0.configs/node/bin/munindoc	2009-11-30 17:41:00.000000000 +0100
@@ -48,7 +48,7 @@
 
 # Dirs in which to look for $plugin.pod and $plugin files.
 my @DIRS = ("$Munin::Common::Defaults::MUNIN_LIBDIR/plugins",
-	    "$Munin::Common::Defaults::MUNIN_CONFDIR/plugins",
+	    "$Munin::Common::Defaults::MUNIN_NCONFDIR/plugins",
             "$Munin::Common::Defaults::MUNIN_PERLLIB");
 
 File::Find::find({wanted => \&wanted_pod}, @DIRS);
diff -ur munin-1.4.0/node/extras/munin-node-simple.in munin-1.4.0.configs/node/extras/munin-node-simple.in
--- munin-1.4.0/node/extras/munin-node-simple.in	2009-11-27 12:17:13.000000000 +0100
+++ munin-1.4.0.configs/node/extras/munin-node-simple.in	2009-11-30 17:41:11.000000000 +0100
@@ -28,8 +28,8 @@
 
 $| = 1;
 
-my $clientdir = "@@CONFDIR@@/node.d";
-my $conffile  = "@@CONFDIR@@/node-simple.conf";
+my $clientdir = "@@NCONFDIR@@/node.d";
+my $conffile  = "@@NCONFDIR@@/node-simple.conf";
 my $version   = "@@VERSION@@";
 
 # Empty environment
diff -ur munin-1.4.0/node/lib/Munin/Node/Config.pm munin-1.4.0.configs/node/lib/Munin/Node/Config.pm
--- munin-1.4.0/node/lib/Munin/Node/Config.pm	2009-11-27 12:17:13.000000000 +0100
+++ munin-1.4.0.configs/node/lib/Munin/Node/Config.pm	2009-11-30 17:40:38.000000000 +0100
@@ -25,7 +25,7 @@
         my ($class) = @_;
         
         $instance ||= bless {
-            config_file  => "$Munin::Common::Defaults::MUNIN_CONFDIR/munin-node.conf",
+            config_file  => "$Munin::Common::Defaults::MUNIN_NCONFDIR/munin-node.conf",
         }, $class;
 
         return $instance;
diff -ur munin-1.4.0/node/lib/Munin/Node/Server.pm munin-1.4.0.configs/node/lib/Munin/Node/Server.pm
--- munin-1.4.0/node/lib/Munin/Node/Server.pm	2009-11-27 12:17:13.000000000 +0100
+++ munin-1.4.0.configs/node/lib/Munin/Node/Server.pm	2009-11-30 17:36:02.000000000 +0100
@@ -227,11 +227,11 @@
     my $mode = $session->{tls_mode};
 
     my $key        = $config->{tls_private_key}
-                  || "$Munin::Common::Defaults::MUNIN_CONFDIR/munin-node.pem";
+                  || "$Munin::Common::Defaults::MUNIN_NCONFDIR/munin-node.pem";
     my $cert       = $config->{tls_certificate}
-                  || "$Munin::Common::Defaults::MUNIN_CONFDIR/munin-node.pem";
+                  || "$Munin::Common::Defaults::MUNIN_NCONFDIR/munin-node.pem";
     my $ca_cert    = $config->{tls_ca_certificate}
-                  || "$Munin::Common::Defaults::MUNIN_CONFDIR/cacert.pem";
+                  || "$Munin::Common::Defaults::MUNIN_NCONFDIR/cacert.pem";
     my $tls_verify = $config->{tls_verify_certificate}
                   || 'no';
 
diff -ur munin-1.4.0/node/sbin/munin-node munin-1.4.0.configs/node/sbin/munin-node
--- munin-1.4.0/node/sbin/munin-node	2009-11-27 12:17:13.000000000 +0100
+++ munin-1.4.0.configs/node/sbin/munin-node	2009-11-30 17:39:16.000000000 +0100
@@ -33,9 +33,9 @@
 use Munin::Node::OS;
 use Munin::Node::Server;
 
-my $servicedir = "$Munin::Common::Defaults::MUNIN_CONFDIR/plugins";
-my $sconfdir   = "$Munin::Common::Defaults::MUNIN_CONFDIR/plugin-conf.d";
-my $conffile   = "$Munin::Common::Defaults::MUNIN_CONFDIR/munin-node.conf";
+my $servicedir = "$Munin::Common::Defaults::MUNIN_NCONFDIR/plugins";
+my $sconfdir   = "$Munin::Common::Defaults::MUNIN_NCONFDIR/plugin-conf.d";
+my $conffile   = "$Munin::Common::Defaults::MUNIN_NCONFDIR/munin-node.conf";
 my $FQDN       = "";
 my $DEBUG      = 0;
 my $PIDEBUG    = 0;
@@ -143,7 +143,7 @@
 Options:
     --help              View this message.
     --config <file>     Use <file> as configuration file.
-                        [$Munin::Common::Defaults::MUNIN_CONFDIR/munin-node.conf]
+                        [$Munin::Common::Defaults::MUNIN_NCONFDIR/munin-node.conf]
     --[no]paranoia      Only run plugins owned by root. Check permissions.
                         [--noparanoia]
     --debug             View debug messages
@@ -180,7 +180,7 @@
 
 =item B<< --config <configfile> >>
 
-Use E<lt>fileE<gt> as configuration file. [@@CONFDIR@@/munin-node.conf]
+Use E<lt>fileE<gt> as configuration file. [@@NCONFDIR@@/munin-node.conf]
 
 =item B< --[no]paranoia >
 
@@ -208,9 +208,9 @@
 
 =head1 FILES
 
-    @@CONFDIR@@/munin-node.conf
-    @@CONFDIR@@/plugins/*
-    @@CONFDIR@@/plugin-conf.d/*
+    @@NCONFDIR@@/munin-node.conf
+    @@NCONFDIR@@/plugins/*
+    @@NCONFDIR@@/plugin-conf.d/*
     @@STATEDIR@@/munin-node.pid
     @@LOGDIR@@/munin-node.log
 
diff -ur munin-1.4.0/node/sbin/munin-node-configure munin-1.4.0.configs/node/sbin/munin-node-configure
--- munin-1.4.0/node/sbin/munin-node-configure	2009-11-27 12:17:13.000000000 +0100
+++ munin-1.4.0.configs/node/sbin/munin-node-configure	2009-11-30 17:34:09.000000000 +0100
@@ -88,9 +88,9 @@
 
 sub parse_args
 {
-	my $conffile   = "$Munin::Common::Defaults::MUNIN_CONFDIR/munin-node.conf";
-	my $servicedir = "$Munin::Common::Defaults::MUNIN_CONFDIR/plugins";
-	my $sconfdir   = "$Munin::Common::Defaults::MUNIN_CONFDIR/plugin-conf.d";
+	my $conffile   = "$Munin::Common::Defaults::MUNIN_NCONFDIR/munin-node.conf";
+	my $servicedir = "$Munin::Common::Defaults::MUNIN_NCONFDIR/plugins";
+	my $sconfdir   = "$Munin::Common::Defaults::MUNIN_NCONFDIR/plugin-conf.d";
 	my $libdir     = "$Munin::Common::Defaults::MUNIN_LIBDIR/plugins";
 
 	my $debug;
@@ -180,11 +180,11 @@
                             printed on STDERR.
 
     --config <file>         Override configuration file
-                                [$Munin::Common::Defaults::MUNIN_CONFDIR/munin.conf]
+                                [$Munin::Common::Defaults::MUNIN_NCONFDIR/munin-node.conf]
     --servicedir <dir>      Override plugin directory
-                                [$Munin::Common::Defaults::MUNIN_CONFDIR/plugins/]
+                                [$Munin::Common::Defaults::MUNIN_NCONFDIR/plugins/]
     --sconfdir <dir>        Override plugin configuration directory
-                                [$Munin::Common::Defaults::MUNIN_CONFDIR/plugin-conf.d]
+                                [$Munin::Common::Defaults::MUNIN_NCONFDIR/plugin-conf.d]
     --libdir <dir>          Override plugin lib
                                 [$Munin::Common::Defaults::MUNIN_LIBDIR/plugins/]
 
@@ -520,15 +520,15 @@
 
 =item B<< --config <file> >>
 
-Override configuration file [@@CONFDIR@@/munin-node.conf]
+Override configuration file [@@NCONFDIR@@/munin-node.conf]
 
 =item B<< --servicedir <dir> >>
 
-Override plugin dir [@@CONFDIR@@/plugins/]
+Override plugin dir [@@NCONFDIR@@/plugins/]
 
 =item B<< --sconfdir <dir> >>
 
-Override plugin configuration directory [@@CONFDIR@@/plugin-conf.d/]
+Override plugin configuration directory [@@NCONFDIR@@/plugin-conf.d/]
 
 =item B<< --libdir <dir> >>
 
@@ -588,9 +588,9 @@
 
 =head1 FILES
 
-    @@CONFDIR@@/munin-node.conf
-    @@CONFDIR@@/plugin-conf.d/*
-    @@CONFDIR@@/plugins/*
+    @@NCONFDIR@@/munin-node.conf
+    @@NCONFDIR@@/plugin-conf.d/*
+    @@NCONFDIR@@/plugins/*
     @@LIBDIR@@/plugins/plugins.history
     @@LIBDIR@@/plugins/*
 
diff -ur munin-1.4.0/node/sbin/munin-run munin-1.4.0.configs/node/sbin/munin-run
--- munin-1.4.0/node/sbin/munin-run	2009-11-27 12:17:13.000000000 +0100
+++ munin-1.4.0.configs/node/sbin/munin-run	2009-11-30 17:40:19.000000000 +0100
@@ -102,9 +102,9 @@
 sub parse_args
 {
     # Default configuration values
-    my $servicedir = "$Munin::Common::Defaults::MUNIN_CONFDIR/plugins";
-    my $sconfdir   = "$Munin::Common::Defaults::MUNIN_CONFDIR/plugin-conf.d";
-    my $conffile   = "$Munin::Common::Defaults::MUNIN_CONFDIR/munin-node.conf";
+    my $servicedir = "$Munin::Common::Defaults::MUNIN_NCONFDIR/plugins";
+    my $sconfdir   = "$Munin::Common::Defaults::MUNIN_NCONFDIR/plugin-conf.d";
+    my $conffile   = "$Munin::Common::Defaults::MUNIN_NCONFDIR/munin-node.conf";
     my $sconffile;
 
     my ($DEBUG, $PIDEBUG) = (0, 0);
@@ -186,11 +186,11 @@
 Options:
     --help              View this message.
     --config <file>     Use <file> as configuration file.
-                        [$Munin::Common::Defaults::MUNIN_CONFDIR/munin-node.conf]
+                        [$Munin::Common::Defaults::MUNIN_NCONFDIR/munin-node.conf]
     --servicedir <dir>  Dir where plugins are found.
-                        [$Munin::Common::Defaults::MUNIN_CONFDIR/plugins]
+                        [$Munin::Common::Defaults::MUNIN_NCONFDIR/plugins]
     --sconfdir <dir>    Dir where plugin configurations are found.
-                        [$Munin::Common::Defaults::MUNIN_CONFDIR/plugin-conf.d]
+                        [$Munin::Common::Defaults::MUNIN_NCONFDIR/plugin-conf.d]
     --sconffile <file>  Use <file> as plugin configuration. Overrides sconfdir.
                         [undefined]
     --[no]paranoia      Only run plugins owned by root. Check permissions.
@@ -226,15 +226,15 @@
 
 =item B<< --config <configfile> >>
 
-Use E<lt>fileE<gt> as configuration file. [@@CONFDIR@@/munin-node.conf]
+Use E<lt>fileE<gt> as configuration file. [@@NCONFDIR@@/munin-node.conf]
 
 =item B<< --servicedir <dir> >>
 
-Use E<lt>dirE<gt> as plugin dir. [@@CONFDIR@@/plugins/]
+Use E<lt>dirE<gt> as plugin dir. [@@NCONFDIR@@/plugins/]
 
 =item B<< --sconfdir <dir> >>
 
-Use E<lt>dirE<gt> as plugin configuration dir. [@@CONFDIR@@/plugin-conf.d/]
+Use E<lt>dirE<gt> as plugin configuration dir. [@@NCONFDIR@@/plugin-conf.d/]
 
 =item B<< --sconffile <file> >>
 
@@ -273,9 +273,9 @@
 
 =head1 FILES
 
-    @@CONFDIR@@/munin-node.conf
-    @@CONFDIR@@/plugins/*
-    @@CONFDIR@@/plugin-conf.d/*
+    @@NCONFDIR@@/munin-node.conf
+    @@NCONFDIR@@/plugins/*
+    @@NCONFDIR@@/plugin-conf.d/*
     @@STATEDIR@@/munin-node.pid
     @@LOGDIR@@/munin-node.log
 
diff -ur munin-1.4.0/plugins/node.d/exim_mailqueue.in munin-1.4.0.configs/plugins/node.d/exim_mailqueue.in
--- munin-1.4.0/plugins/node.d/exim_mailqueue.in	2009-11-27 12:17:10.000000000 +0100
+++ munin-1.4.0.configs/plugins/node.d/exim_mailqueue.in	2009-11-30 17:42:09.000000000 +0100
@@ -15,7 +15,7 @@
   [exim_*]
     user exim
 
-Configuration parameters for a file in @@CONFDIR@@/plugin-conf.d/
+Configuration parameters for a file in @@NCONFDIR@@/plugin-conf.d/
 if you need to override the defaults below:
 
   [exim_mailqueue]
diff -ur munin-1.4.0/plugins/node.d/ipmi_sensor_.in munin-1.4.0.configs/plugins/node.d/ipmi_sensor_.in
--- munin-1.4.0/plugins/node.d/ipmi_sensor_.in	2009-11-27 12:17:11.000000000 +0100
+++ munin-1.4.0.configs/plugins/node.d/ipmi_sensor_.in	2009-11-30 17:42:27.000000000 +0100
@@ -17,7 +17,7 @@
 ipmitool probably needs to be run as root, and it may take more than
 10 seconds on some hosts.
 
-Add the following to your @@CONFDIR@@/munin-node:
+Add the following to your @@NCONFDIR@@/munin-node:
 
  [ipmi_sensor_*]
   user root
@@ -50,7 +50,7 @@
 
 =head1 BUGS
 
-Plugin reads @@CONFDIR@@/ipmi directly, instead of reading environment
+Plugin reads @@NCONFDIR@@/ipmi directly, instead of reading environment
 variables.
 
 =head1 MAGIC MARKERS
@@ -73,7 +73,7 @@
 CACHEDIR = "@@PLUGSTATE@@"
 CACHEFILE = "plugin-ipmi_sensor.cache"
 CACHEAGE = 120
-CONFIG = '@@CONFDIR@@/ipmi'
+CONFIG = '@@NCONFDIR@@/ipmi'
 
 
 
diff -ur munin-1.4.0/plugins/node.d/mysql_innodb.in munin-1.4.0.configs/plugins/node.d/mysql_innodb.in
--- munin-1.4.0/plugins/node.d/mysql_innodb.in	2009-11-27 12:17:11.000000000 +0100
+++ munin-1.4.0.configs/plugins/node.d/mysql_innodb.in	2009-11-30 17:42:53.000000000 +0100
@@ -34,7 +34,7 @@
 
 =head1 CONFIGURATION
 
-Configuration parameters for @@CONFDIR@@/mysql_innodb,
+Configuration parameters for @@NCONFDIR@@/mysql_innodb,
 if you need to override the defaults below:
 
  [mysql_innodb]
diff -ur munin-1.4.0/plugins/node.d/mysql_isam_space_.in munin-1.4.0.configs/plugins/node.d/mysql_isam_space_.in
--- munin-1.4.0/plugins/node.d/mysql_isam_space_.in	2009-11-27 12:17:10.000000000 +0100
+++ munin-1.4.0.configs/plugins/node.d/mysql_isam_space_.in	2009-11-30 17:41:30.000000000 +0100
@@ -11,7 +11,7 @@
 
 =head1 CONFIGURATION
 
-Configuration parameters for @@CONFDIR@@/PLUGIN,
+Configuration parameters for @@NCONFDIR@@/PLUGIN,
 if you need to override the defaults below:
 
  [mysql_isam_space_*]
diff -ur munin-1.4.0/plugins/node.d/mysql_threads.in munin-1.4.0.configs/plugins/node.d/mysql_threads.in
--- munin-1.4.0/plugins/node.d/mysql_threads.in	2009-11-30 17:45:38.180724509 +0100
+++ munin-1.4.0.configs/plugins/node.d/mysql_threads.in	2009-11-30 17:43:19.000000000 +0100
@@ -9,7 +9,7 @@
 
 =head1 CONFIGURATION
 
-Configuration parameters for @@CONFDIR@@/mysql_threads, if you
+Configuration parameters for @@NCONFDIR@@/mysql_threads, if you
 need to override the defaults below:
 
  [mysql_threads]
diff -ur munin-1.4.0/plugins/node.d/named.in munin-1.4.0.configs/plugins/node.d/named.in
--- munin-1.4.0/plugins/node.d/named.in	2009-11-27 12:17:11.000000000 +0100
+++ munin-1.4.0.configs/plugins/node.d/named.in	2009-11-30 17:43:03.000000000 +0100
@@ -34,7 +34,7 @@
 
 =head1 CONFIGURATION
 
-Configuration parameters for @@CONFDIR@@/named,
+Configuration parameters for @@NCONFDIR@@/named,
 if you need to override the defaults below:
 
  [named]
diff -ur munin-1.4.0/plugins/node.d/ntp_kernel_pll_freq.in munin-1.4.0.configs/plugins/node.d/ntp_kernel_pll_freq.in
--- munin-1.4.0/plugins/node.d/ntp_kernel_pll_freq.in	2009-11-27 12:17:10.000000000 +0100
+++ munin-1.4.0.configs/plugins/node.d/ntp_kernel_pll_freq.in	2009-11-30 17:41:46.000000000 +0100
@@ -12,7 +12,7 @@
 
 No configuration
 
-This plugin optionally reads the file @@CONFDIR@@/ntp-freq-comp, which
+This plugin optionally reads the file @@NCONFDIR@@/ntp-freq-comp, which
 should contain a number to be added to the frequency read by ntpdc.
 
 =head1 AUTHORS
@@ -46,8 +46,8 @@
     exit 0
 fi
 
-if [ -f @@CONFDIR@@/ntp-freq-comp ]; then
-    fcomp=`cat @@CONFDIR@@/ntp-freq-comp`
+if [ -f @@NCONFDIR@@/ntp-freq-comp ]; then
+    fcomp=`cat @@NCONFDIR@@/ntp-freq-comp`
 else
     fcomp=0
 fi
diff -ur munin-1.4.0/plugins/node.d/postfix_mailstats.in munin-1.4.0.configs/plugins/node.d/postfix_mailstats.in
--- munin-1.4.0/plugins/node.d/postfix_mailstats.in	2009-11-27 12:17:10.000000000 +0100
+++ munin-1.4.0.configs/plugins/node.d/postfix_mailstats.in	2009-11-30 17:42:45.000000000 +0100
@@ -8,7 +8,7 @@
 
 =head1 CONFIGURATION
 
-Configuration parameters for @@CONFDIR@@/postfix_mailstats,
+Configuration parameters for @@NCONFDIR@@/postfix_mailstats,
 if you need to override the defaults below:
 
  [postfix_mailstats]
diff -ur munin-1.4.0/plugins/node.d/psu_.in munin-1.4.0.configs/plugins/node.d/psu_.in
--- munin-1.4.0/plugins/node.d/psu_.in	2009-11-27 12:17:10.000000000 +0100
+++ munin-1.4.0.configs/plugins/node.d/psu_.in	2009-11-30 17:42:37.000000000 +0100
@@ -15,7 +15,7 @@
 
 Example:
 
- ln -s @@LIBDIR@@/plugins/psu_ @@CONFDIR@@/plugins/psu_foo
+ ln -s @@LIBDIR@@/plugins/psu_ @@NCONFDIR@@/plugins/psu_foo
 
 ...will monitor the user "foo"
 
diff -ur munin-1.4.0/plugins/node.d/squeezebox_.in munin-1.4.0.configs/plugins/node.d/squeezebox_.in
--- munin-1.4.0/plugins/node.d/squeezebox_.in	2009-11-27 12:17:10.000000000 +0100
+++ munin-1.4.0.configs/plugins/node.d/squeezebox_.in	2009-11-30 17:43:10.000000000 +0100
@@ -19,7 +19,7 @@
 non-default configuration, please use the environment variables
 'squeezebox_host' and 'squeezebox_port' to connect. Also, if your
 netcat(1) binary is anywhere else than /bin/nc please define it in the
-plugin's environment file (@@CONFDIR@@/squeezebox or
+plugin's environment file (@@NCONFDIR@@/squeezebox or
 similar). Sample follows:
 
  [squeezebox_*]
diff -ur munin-1.4.0/plugins/node.d/varnish_.in munin-1.4.0.configs/plugins/node.d/varnish_.in
--- munin-1.4.0/plugins/node.d/varnish_.in	2009-11-27 12:17:10.000000000 +0100
+++ munin-1.4.0.configs/plugins/node.d/varnish_.in	2009-11-30 17:41:59.000000000 +0100
@@ -54,7 +54,7 @@
 You can link them yourself with something like this:
 
   ln -s @@LIBDIR@@/plugins/varnish_ \
-    @@CONFDIR@@/plugins/varnish_data_structures
+    @@NCONFDIR@@/plugins/varnish_data_structures
 
 =head1 INTERPERTATION
 
diff -ur munin-1.4.0/plugins/node.d.linux/acpi.in munin-1.4.0.configs/plugins/node.d.linux/acpi.in
--- munin-1.4.0/plugins/node.d.linux/acpi.in	2009-11-27 12:17:05.000000000 +0100
+++ munin-1.4.0.configs/plugins/node.d.linux/acpi.in	2009-11-30 17:43:46.000000000 +0100
@@ -17,7 +17,7 @@
 
 =head1 USAGE
 
-Link this plugin to @@CONFDIR@@/plugins/ and restart the munin-node.
+Link this plugin to @@NCONFDIR@@/plugins/ and restart the munin-node.
 
 =head1 INTERPRETATION
 
diff -ur munin-1.4.0/plugins/node.d.linux/apt.in munin-1.4.0.configs/plugins/node.d.linux/apt.in
--- munin-1.4.0/plugins/node.d.linux/apt.in	2009-11-27 12:17:05.000000000 +0100
+++ munin-1.4.0.configs/plugins/node.d.linux/apt.in	2009-11-30 17:44:34.000000000 +0100
@@ -38,7 +38,7 @@
 
 =head1 USAGE
 
-Link this plugin in @@CONFDIR@@/plugins/ and restart the munin-node.
+Link this plugin in @@NCONFDIR@@/plugins/ and restart the munin-node.
 
 =head1 MAGIC MARKERS
 
diff -ur munin-1.4.0/plugins/node.d.linux/df.in munin-1.4.0.configs/plugins/node.d.linux/df.in
--- munin-1.4.0/plugins/node.d.linux/df.in	2009-11-27 12:17:05.000000000 +0100
+++ munin-1.4.0.configs/plugins/node.d.linux/df.in	2009-11-30 17:44:11.000000000 +0100
@@ -28,11 +28,11 @@
     env.warning 92
     env.critical 98
 
-Put it in a file in @@CONFDIR@@/plugin-conf.d/ and restart the munin-node.
+Put it in a file in @@NCONFDIR@@/plugin-conf.d/ and restart the munin-node.
 
 =head1 USAGE 
 
-Link this plugin to @@CONFDIR@@/plugins/ and restart the munin-node.
+Link this plugin to @@NCONFDIR@@/plugins/ and restart the munin-node.
 
 =head1 MAGIC MARKERS
 
diff -ur munin-1.4.0/plugins/node.d.linux/fw_conntrack.in munin-1.4.0.configs/plugins/node.d.linux/fw_conntrack.in
--- munin-1.4.0/plugins/node.d.linux/fw_conntrack.in	2009-11-27 12:17:05.000000000 +0100
+++ munin-1.4.0.configs/plugins/node.d.linux/fw_conntrack.in	2009-11-30 17:44:18.000000000 +0100
@@ -14,7 +14,7 @@
 
 =head2 CONFIGURATION EXAMPLE
 
-@@CONFDIR@@/plugin-conf.d/global or other file in that dir must contain:
+@@NCONFDIR@@/plugin-conf.d/global or other file in that dir must contain:
 
  [fw*]
   user root
diff -ur munin-1.4.0/plugins/node.d.linux/fw_forwarded_local.in munin-1.4.0.configs/plugins/node.d.linux/fw_forwarded_local.in
--- munin-1.4.0/plugins/node.d.linux/fw_forwarded_local.in	2009-11-27 12:17:05.000000000 +0100
+++ munin-1.4.0.configs/plugins/node.d.linux/fw_forwarded_local.in	2009-11-30 17:43:36.000000000 +0100
@@ -13,7 +13,7 @@
 
 =head1 CONFIGURATION EXAMPLE
 
-@@CONFDIR@@/plugin-conf.d/global or other file in that dir must contain:
+@@NCONFDIR@@/plugin-conf.d/global or other file in that dir must contain:
 
  [fw*]
   user root
diff -ur munin-1.4.0/plugins/node.d.linux/fw_packets.in munin-1.4.0.configs/plugins/node.d.linux/fw_packets.in
--- munin-1.4.0/plugins/node.d.linux/fw_packets.in	2009-11-27 12:17:05.000000000 +0100
+++ munin-1.4.0.configs/plugins/node.d.linux/fw_packets.in	2009-11-30 17:43:55.000000000 +0100
@@ -11,7 +11,7 @@
 
 =head1 CONFIGURATION EXAMPLE
 
-@@CONFDIR@@/plugin-conf.d/global or other file in that dir must contain:
+@@NCONFDIR@@/plugin-conf.d/global or other file in that dir must contain:
 
  [fw*]
   user root
diff -ur munin-1.4.0/plugins/node.d.linux/iostat.in munin-1.4.0.configs/plugins/node.d.linux/iostat.in
--- munin-1.4.0/plugins/node.d.linux/iostat.in	2009-11-30 17:45:38.201723751 +0100
+++ munin-1.4.0.configs/plugins/node.d.linux/iostat.in	2009-11-30 17:44:41.000000000 +0100
@@ -24,7 +24,7 @@
 
 =head1 USAGE
 
-Link this plugin to @@CONFDIR@@/plugins/ and restart the munin-node.
+Link this plugin to @@NCONFDIR@@/plugins/ and restart the munin-node.
 
 =head1 INTERPRETATION
 
diff -ur munin-1.4.0/plugins/node.d.linux/iostat_ios.in munin-1.4.0.configs/plugins/node.d.linux/iostat_ios.in
--- munin-1.4.0/plugins/node.d.linux/iostat_ios.in	2009-11-27 12:17:05.000000000 +0100
+++ munin-1.4.0.configs/plugins/node.d.linux/iostat_ios.in	2009-11-30 17:44:26.000000000 +0100
@@ -15,7 +15,7 @@
 
 =head1 USAGE
 
-Link this into @@CONFDIR@@/plugins/ and restart the munin-node.
+Link this into @@NCONFDIR@@/plugins/ and restart the munin-node.
 
 =head1 INTERPRETATION
 
diff -ur munin-1.4.0/plugins/node.d.linux/irqstats.in munin-1.4.0.configs/plugins/node.d.linux/irqstats.in
--- munin-1.4.0/plugins/node.d.linux/irqstats.in	2009-11-27 12:17:05.000000000 +0100
+++ munin-1.4.0.configs/plugins/node.d.linux/irqstats.in	2009-11-30 17:44:02.000000000 +0100
@@ -15,7 +15,7 @@
 
 =head1 USAGE
 
-Link this plugin to @@CONFDIR@@/plugins/ and restart the munin-node.
+Link this plugin to @@NCONFDIR@@/plugins/ and restart the munin-node.
 
 =head1 INTERPRETATION
 
diff -ur munin-1.4.0/resources/hpux-init.d_munin-node.in munin-1.4.0.configs/resources/hpux-init.d_munin-node.in
--- munin-1.4.0/resources/hpux-init.d_munin-node.in	2009-11-27 12:17:18.000000000 +0100
+++ munin-1.4.0.configs/resources/hpux-init.d_munin-node.in	2009-11-30 17:45:11.000000000 +0100
@@ -6,7 +6,7 @@
 PATH=/sbin:/bin:/usr/sbin:/usr/bin
 NAME=munin-node
 DAEMON=@@SBINDIR@@/$NAME
-CONFFILE=@@CONFDIR@@/munin-node.conf
+CONFFILE=@@NCONFDIR@@/munin-node.conf
 RUN_AS=$(awk '$1~/user/{print$2}' $CONFFILE)
 PIDFILE=$(awk '$1~/pid/{print$2}' $CONFFILE)
 MYCONF=/etc/rc.config.d/muninconf
diff -ur munin-1.4.0/resources/linux-cron.d_munin-node.in munin-1.4.0.configs/resources/linux-cron.d_munin-node.in
--- munin-1.4.0/resources/linux-cron.d_munin-node.in	2009-11-27 12:17:18.000000000 +0100
+++ munin-1.4.0.configs/resources/linux-cron.d_munin-node.in	2009-11-30 17:45:01.000000000 +0100
@@ -8,5 +8,5 @@
 # an hour (12 invokations an hour, 1 in 12 chance that the update will
 # happen), but ensure that there will never be more than two hour (7200
 # seconds) interval between updates..
-*/5 * * * *	root [ -x @@CONFDIR@@/plugins/apt ] && @@CONFDIR@@/plugins/apt update 7200 12 >/dev/null
+*/5 * * * *	root [ -x @@NCONFDIR@@/plugins/apt ] && @@NCONFDIR@@/plugins/apt update 7200 12 >/dev/null
 
diff -ur munin-1.4.0/t/Makefile.config munin-1.4.0.configs/t/Makefile.config
--- munin-1.4.0/t/Makefile.config	2009-11-27 12:17:18.000000000 +0100
+++ munin-1.4.0.configs/t/Makefile.config	2009-11-30 17:45:21.000000000 +0100
@@ -9,7 +9,8 @@
 PREFIX     = $(PWD)/t/install
 
 # Where Munin keeps its configurations (server.conf, client.conf, ++)
-CONFDIR    = $(PREFIX)/etc
+MCONFDIR    = $(PREFIX)/etc
+NCONFDIR    = $(PREFIX)/etc
 
 # Client only - Where to put RRD files and other intenal data
 DBDIR      = $(PREFIX)/var/lib
