diff -Naur munin-1.3.4/dists/tarball/plugins.conf munin-1.3.4-p/dists/tarball/plugins.conf
--- munin-1.3.4/dists/tarball/plugins.conf	Sat Mar  8 02:42:39 2008
+++ munin-1.3.4-p/dists/tarball/plugins.conf	Fri Apr  4 11:13:06 2008
@@ -31,3 +31,74 @@
 
 [postfix*]
 user root
+env.logdir /var/log
+env.logfile maillog
+
+[netstat]
+group proc
+
+[amavis]
+user root
+env.amavislog /var/log/maillog
+
+[courier*]
+user root
+env.logfile /var/log/maillog
+
+[mailman]
+user mailman
+group mailman
+
+[bind9_rndc]
+group named
+env.querystats /var/lib/named/named.stats
+
+[squid*]
+group squid
+#env.squidport 3128
+#env.squiduser manager
+#env.squidpasswd <SECRET>
+
+[fw*]
+group proc
+
+[if*]
+group proc
+
+[dhcpd3]
+user root
+
+[ip_*]
+user root
+
+[munin*]
+user munin
+
+[ping_*]
+group adm
+
+[port_*]
+group proc
+
+[processes]
+group proc
+
+[ps_*]
+group proc
+
+[psu_*]
+group proc
+
+[nfs*]
+group proc
+
+[entropy]
+group proc
+
+[open_*]
+group proc
+
+[nut*]
+env.upsname SmartUPS
+env.upshost 127.0.0.1
+
diff -Naur munin-1.3.4/node/node.d/apache_accesses.in munin-1.3.4-p/node/node.d/apache_accesses.in
--- munin-1.3.4/node/node.d/apache_accesses.in	Sat Mar  8 02:42:39 2008
+++ munin-1.3.4-p/node/node.d/apache_accesses.in	Fri Apr  4 11:13:06 2008
@@ -86,6 +86,7 @@
 
 my $URL = exists $ENV{'url'} ? $ENV{'url'} : "http://127.0.0.1:%d/server-status?auto";
 my @PORTS = exists $ENV{'ports'} ? split(' ', $ENV{'ports'}) : (80);
+my $HOST = exists $ENV{'host'} ? $ENV{'host'} : undef;
 
 if ( defined $ARGV[0] and $ARGV[0] eq "autoconf" )
 {
@@ -114,7 +115,11 @@
 
 if ( defined $ARGV[0] and $ARGV[0] eq "config" )
 {
-	print "graph_title Apache accesses\n";
+	print "graph_title Apache accesses";
+	if ( $HOST ) {
+		print " for $HOST";
+	}
+	print "\n";
 	print "graph_args --base 1000\n";
 	print "graph_vlabel accesses / \${graph_period}\n";
 	print "graph_category apache\n";
diff -Naur munin-1.3.4/node/node.d/apache_processes.in munin-1.3.4-p/node/node.d/apache_processes.in
--- munin-1.3.4/node/node.d/apache_processes.in	Sat Mar  8 02:42:39 2008
+++ munin-1.3.4-p/node/node.d/apache_processes.in	Fri Apr  4 11:13:06 2008
@@ -88,6 +88,7 @@
 
 my $URL = exists $ENV{'url'} ? $ENV{'url'} : "http://127.0.0.1:%d/server-status?auto";
 my @PORTS = exists $ENV{'ports'} ? split(' ', $ENV{'ports'}) : (80);
+my $HOST = exists $ENV{'host'} ? $ENV{'host'} : undef;
 
 if ( defined $ARGV[0] and $ARGV[0] eq "autoconf" )
 {
@@ -119,7 +120,11 @@
 
 if ( defined $ARGV[0] and $ARGV[0] eq "config" )
 {
-        print "graph_title Apache processes\n";
+        print "graph_title Apache processes";
+	if ( $HOST ) {
+		print " for $HOST";
+	}
+	print "\n";
         print "graph_args --base 1000 -l 0\n";
 		print "graph_category apache\n";
 	print "graph_order ";
diff -Naur munin-1.3.4/node/node.d/apache_volume.in munin-1.3.4-p/node/node.d/apache_volume.in
--- munin-1.3.4/node/node.d/apache_volume.in	Sat Mar  8 02:42:39 2008
+++ munin-1.3.4-p/node/node.d/apache_volume.in	Fri Apr  4 11:13:06 2008
@@ -86,6 +86,7 @@
 
 my $URL = exists $ENV{'url'} ? $ENV{'url'} : "http://127.0.0.1:%d/server-status?auto";
 my @PORTS = exists $ENV{'ports'} ? split(' ', $ENV{'ports'}) : (80);
+my $HOST = exists $ENV{'host'} ? $ENV{'host'} : undef;
 
 if ( defined $ARGV[0] and $ARGV[0] eq "autoconf" )
 {
@@ -114,7 +115,11 @@
 
 if ( defined $ARGV[0] and $ARGV[0] eq "config" )
 {
-	print "graph_title Apache volume\n";
+	print "graph_title Apache volume";
+	if ( $HOST ) {
+		print " for $HOST";
+	}
+	print "\n";
 	print "graph_args --base 1000\n";
 	print "graph_vlabel bytes per \${graph_period}\n";
 	print "graph_category apache\n";
diff -Naur munin-1.3.4/node/node.d/bind9.in munin-1.3.4-p/node/node.d/bind9.in
--- munin-1.3.4/node/node.d/bind9.in	Sat Mar  8 02:42:39 2008
+++ munin-1.3.4-p/node/node.d/bind9.in	Fri Apr  4 11:13:06 2008
@@ -19,7 +19,7 @@
 my %IN;
 
 sub get_state {
-    open(Q,"< $STATEFILE") or die;
+    open(Q,"< $STATEFILE") or return;
     while (<Q>) {
         chomp;
         my ($q,$n) = split(/\s+/,$_,2);
diff -Naur munin-1.3.4/node/node.d/courier_.in munin-1.3.4-p/node/node.d/courier_.in
--- munin-1.3.4/node/node.d/courier_.in	Sat Mar  8 02:42:39 2008
+++ munin-1.3.4-p/node/node.d/courier_.in	Fri Apr  4 11:13:06 2008
@@ -35,7 +35,7 @@
 COURIER_LOG=${logfile:-/var/log/mail.log}
 SERVICE=${service:-`basename $0 | sed 's/^courier_//g'`}
 OFFSET_FILE=@@PLUGSTATE@@/courier_${SERVICE}.offset
-LOGTAIL=${logtail:-/usr/sbin/logtail}
+LOGTAIL=${logtail:-`which logtail`}
 
 mktempfile () {
 @@MKTEMP@@
diff -Naur munin-1.3.4/node/node.d/courier_mta_mailstats.in munin-1.3.4-p/node/node.d/courier_mta_mailstats.in
--- munin-1.3.4/node/node.d/courier_mta_mailstats.in	Sat Mar  8 02:42:39 2008
+++ munin-1.3.4-p/node/node.d/courier_mta_mailstats.in	Fri Apr  4 11:13:06 2008
@@ -101,7 +101,9 @@
 
 if($startsize < $pos) {
     # Log rotated
-    parseLogfile ($rotlogfile, $pos, (stat $rotlogfile)[7]);
+    if (-f $rotlogfile) {
+        parseLogfile ($rotlogfile, $pos, (stat $rotlogfile)[7]);
+    }
     $pos = 0;
 }
 
diff -Naur munin-1.3.4/node/node.d/courier_mta_mailvolume.in munin-1.3.4-p/node/node.d/courier_mta_mailvolume.in
--- munin-1.3.4/node/node.d/courier_mta_mailvolume.in	Sat Mar  8 02:42:39 2008
+++ munin-1.3.4-p/node/node.d/courier_mta_mailvolume.in	Fri Apr  4 11:13:06 2008
@@ -92,7 +92,9 @@
 
 if($startsize < $pos) {
     # Log rotated
-    parseLogfile ($rotlogfile, $pos, (stat $rotlogfile)[7]);
+    if (-f $rotlogfile) {
+        parseLogfile ($rotlogfile, $pos, (stat $rotlogfile)[7]);
+    }
     $pos = 0;
 }
 
diff -Naur munin-1.3.4/node/node.d/cupsys_pages.in munin-1.3.4-p/node/node.d/cupsys_pages.in
--- munin-1.3.4/node/node.d/cupsys_pages.in	Sat Mar  8 02:42:39 2008
+++ munin-1.3.4-p/node/node.d/cupsys_pages.in	Fri Apr  4 11:13:06 2008
@@ -94,7 +94,9 @@
 }
 if ($startsize < $pos) {
     # Log rotated
-    parselogfile ($rotlogfile, $pos, (stat $rotlogfile)[7]);
+    if (-f $rotlogfile) {
+        parselogfile ($rotlogfile, $pos, (stat $rotlogfile)[7]);
+    }
     $pos = 0;
 }
 
diff -Naur munin-1.3.4/node/node.d/mailman.in munin-1.3.4-p/node/node.d/mailman.in
--- munin-1.3.4/node/node.d/mailman.in	Sat Mar  8 02:42:39 2008
+++ munin-1.3.4-p/node/node.d/mailman.in	Fri Apr  4 11:13:06 2008
@@ -69,7 +69,9 @@
 
 if($startsize < $pos) {
     # Log rotated
-    parseMailmanLog($rotlogfile, $pos, (stat $rotlogfile)[7]);
+    if (-f $rotlogfile) {
+        parseMailmanLog($rotlogfile, $pos, (stat $rotlogfile)[7]);
+    }
     $pos = 0;
 }
 
diff -Naur munin-1.3.4/node/node.d/mysql_slowqueries.in munin-1.3.4-p/node/node.d/mysql_slowqueries.in
--- munin-1.3.4/node/node.d/mysql_slowqueries.in	Sat Mar  8 02:42:39 2008
+++ munin-1.3.4-p/node/node.d/mysql_slowqueries.in	Fri Apr  4 11:13:06 2008
@@ -47,5 +47,5 @@
 	exit 0
 fi
 
-/usr/bin/printf "queries.value "
+/bin/printf "queries.value "
 ($MYSQLADMIN $MYSQLOPTS status 2>/dev/null || echo a a a a a a a a U) | awk '{print $9}'
diff -Naur munin-1.3.4/node/node.d/mysql_threads.in munin-1.3.4-p/node/node.d/mysql_threads.in
--- munin-1.3.4/node/node.d/mysql_threads.in	Sat Mar  8 02:42:39 2008
+++ munin-1.3.4-p/node/node.d/mysql_threads.in	Fri Apr  4 11:13:06 2008
@@ -45,5 +45,5 @@
 	exit 0
 fi
 
-/usr/bin/printf "threads.value "
+/bin/printf "threads.value "
 ($MYSQLADMIN $MYSQLOPTS status 2>/dev/null || echo 'a a a U') | awk '{print $4}'
diff -Naur munin-1.3.4/node/node.d/ntp_.in munin-1.3.4-p/node/node.d/ntp_.in
--- munin-1.3.4/node/node.d/ntp_.in	Sat Mar  8 02:42:39 2008
+++ munin-1.3.4-p/node/node.d/ntp_.in	Fri Apr  4 11:13:06 2008
@@ -32,9 +32,9 @@
 my $nodelay = $ENV{'nodelay'} || 0;
 
 if ($ARGV[0] and $ARGV[0] eq "autoconf") {
-	`ntpq -c help >/dev/null 2>/dev/null`;
+	`/usr/sbin/ntpq -c help >/dev/null 2>/dev/null`;
 	if ($? eq "0") {
-		if (`ntpq -c "hostnames no" -c peers | wc -l` > 0) {
+		if (`/usr/sbin/ntpq -c "hostnames no" -c peers | wc -l` > 0) {
 			print "yes\n";
 			exit 0;
 		} else {
@@ -42,13 +42,13 @@
 			exit 1;
 		}
 	} else {
-		print "no (ntpq not found)\n";
+		print "no (/usr/sbin/ntpq not found)\n";
 		exit 1;
 	}
 }
 
 if ($ARGV[0] and $ARGV[0] eq "suggest") {
-	my @lines = `ntpq -c "hostnames no" -c peers`;
+	my @lines = `/usr/sbin/ntpq -c "hostnames no" -c peers`;
 	foreach (@lines) {
 		next unless /^.(\d+\.\d+\.\d+\.\d+)/;
 		next if /^.224\.0\.1\.1/;
@@ -73,7 +73,7 @@
 exit 2 unless defined $name;
 
 if ($ARGV[0] and $ARGV[0] eq "config") {
-	my @lines = `ntpq -c "hostnames no" -c peers`;
+	my @lines = `/usr/sbin/ntpq -c "hostnames no" -c peers`;
 	my $host;
 	foreach (@lines) {
 		next unless /^.(\d+\.\d+\.\d+\.\d+)/;
@@ -108,7 +108,7 @@
         exit 0;
 }
 
-my @lines = `ntpq -c "hostnames no" -c peers`;
+my @lines = `/usr/sbin/ntpq -c "hostnames no" -c peers`;
 foreach (@lines) {
 	next unless /^.(\d+\.\d+\.\d+\.\d+)/;
 	next if /^.224\.0\.1\.1/;
diff -Naur munin-1.3.4/node/node.d/ntp_states.in munin-1.3.4-p/node/node.d/ntp_states.in
--- munin-1.3.4/node/node.d/ntp_states.in	Sat Mar  8 02:42:39 2008
+++ munin-1.3.4-p/node/node.d/ntp_states.in	Fri Apr  4 11:13:06 2008
@@ -35,9 +35,9 @@
                );
 
 if ($ARGV[0] and $ARGV[0] eq "autoconf") {
-	`ntpq -c help >/dev/null 2>/dev/null`;
+	`/usr/sbin/ntpq -c help >/dev/null 2>/dev/null`;
 	if ($? eq "0") {
-		if (`ntpq -c "hostnames no" -c peers | wc -l` > 0) {
+		if (`/usr/sbin/ntpq -c "hostnames no" -c peers | wc -l` > 0) {
 			print "yes\n";
 			exit 0;
 		} else {
@@ -45,7 +45,7 @@
 			exit 0;
 		}
 	} else {
-		print "no (ntpq not found)\n";
+		print "no (/usr/sbin/ntpq not found)\n";
 		exit 1;
 	}
 }
@@ -54,7 +54,7 @@
         print "graph_title NTP states\n";
 	print "graph_args --base 1000 --vertical-label msec --lower-limit 0\n";
 	print "graph_category time\n";
-	foreach (`ntpq -c "hostnames no" -c peers`) {
+	foreach (`/usr/sbin/ntpq -c "hostnames no" -c peers`) {
 		next unless /^.(\d+\.\d+\.\d+\.\d+)/;
 		next if /^.224\.0\.1\.1/;
 		my $addr = $1;
@@ -75,7 +75,7 @@
         exit 0;
 }
 
-foreach (`ntpq -c "hostnames no" -c peers`) {
+foreach (`/usr/sbin/ntpq -c "hostnames no" -c peers`) {
 	next unless /^(.)(\d+\.\d+\.\d+\.\d+)/;
 	next if /^.224\.0\.1\.1/;
 	my $state = $1;
diff -Naur munin-1.3.4/node/node.d/nut_misc.in munin-1.3.4-p/node/node.d/nut_misc.in
--- munin-1.3.4/node/node.d/nut_misc.in	Sat Mar  8 02:42:39 2008
+++ munin-1.3.4-p/node/node.d/nut_misc.in	Fri Apr  4 11:13:06 2008
@@ -1,12 +1,13 @@
-!@@PERL@@
+#!@@PERL@@
 
 use strict;
 
 my %status;
 
 my %config = (
-	upsname => 'bertha@127.0.0.1',
-	upsc => 'upsc'
+	upsname => $ENV{upsname} || 'bertha',
+	upshost => $ENV{upshost} || '127.0.0.1',
+	upsc => $ENV{upsc} || 'upsc'
 );
 
 my %graph =  (
@@ -15,11 +16,6 @@
 				type => 'GAUGE',
 				draw => 'LINE2'
 			 },
-	'battery_voltage' => {
-				label => 'batt voltage - V',
-				type => 'GAUGE',
-				draw => 'LINE2'
-			 },
 	'battery_runtime' => {
 				label => 'runtime - min',
 				type => 'GAUGE',
@@ -43,7 +39,12 @@
 );
 
 if ( defined $ARGV[0] and $ARGV[0] eq 'config' ) {
-	print "graph_title UPS Misc - $config{upsname}\n";
+	my $s = $config{upsname};
+	$s =~ s/_/ /g;
+	if ($config{upshost} ne "127.0.0.1" && $config{upshost} ne "localhost") {
+		$s .= " at $config{upshost}";
+	}
+	print "graph_title UPS Misc - $s\n";
 	#print "graph_args -l 110\n";
 	#print "graph_vlabel Misc\n";
 	foreach my $key (keys %graph) {
@@ -56,7 +57,7 @@
 }
 
 sub fetch_values {
-	my $data = `$config{upsc} $config{upsname}`;
+	my $data = `$config{upsc} $config{upsname}\@$config{upshost}`;
 	while ($data =~ /([a-z.]+): (.+)\b/g) {
 		my $label = $1;
 		my $value = $2;
diff -Naur munin-1.3.4/node/node.d/nut_volts.in munin-1.3.4-p/node/node.d/nut_volts.in
--- munin-1.3.4/node/node.d/nut_volts.in	Sat Mar  8 02:42:39 2008
+++ munin-1.3.4-p/node/node.d/nut_volts.in	Fri Apr  4 11:13:06 2008
@@ -5,8 +5,9 @@
 my %status;
 
 my %config = (
-	upsname => 'bertha@127.0.0.1',
-	upsc => 'upsc'
+	upsname => $ENV{upsname} || 'bertha',
+	upshost => $ENV{upshost} || '127.0.0.1',
+	upsc => $ENV{upsc} || 'upsc'
 );
 
 my %graph =  (
@@ -15,16 +16,6 @@
 				type => 'GAUGE',
 				draw => 'LINE2'
 			 },
-	'input_voltage_maximum' => {
-				label => 'max input seen',
-				type => 'GAUGE',
-				draw => 'LINE1'
-			 },
-	'input_voltage_minimum' => {
-				label => 'min input seen',
-				type => 'GAUGE',
-				draw => 'LINE1'
-			 },
 	'output_voltage' => {
 				label => 'output',
 				type => 'GAUGE',
@@ -33,7 +24,12 @@
 );
 
 if ( defined $ARGV[0] and $ARGV[0] eq 'config' ) {
-	print "graph_title UPS Voltages - $config{upsname}\n";
+	my $s = $config{upsname};
+	$s =~ s/_/ /g;
+	if ($config{upshost} ne "127.0.0.1" && $config{upshost} ne "localhost") {
+		$s .= " at $config{upshost}";
+	}
+	print "graph_title UPS Voltages - $s\n";
 	print "graph_args -l 115\n";
 	print "graph_vlabel Volts\n";
 	foreach my $key (keys %graph) {
@@ -46,7 +42,7 @@
 }
 
 sub fetch_values {
-	my $data = `$config{upsc} $config{upsname}`;
+	my $data = `$config{upsc} $config{upsname}\@$config{upshost}`;
 	while ($data =~ /([a-z.]+): (.+)\b/g) {
 		my $label = $1;
 		my $value = $2;
diff -Naur munin-1.3.4/node/node.d/perdition.in munin-1.3.4-p/node/node.d/perdition.in
--- munin-1.3.4/node/node.d/perdition.in	Sat Mar  8 02:42:39 2008
+++ munin-1.3.4-p/node/node.d/perdition.in	Fri Apr  4 11:13:06 2008
@@ -30,7 +30,7 @@
 # Set the location of the perdition logs
 PERDITION_LOG=${logfile:-/var/log/perdition.log}
 OFFSET_FILE=@@PLUGSTATE@@/perdition.offset
-LOGTAIL=${logtail:-/usr/sbin/logtail}
+LOGTAIL=${logtail:-`which logtail`}
 
 case $1 in
     autoconf|detect)
diff -Naur munin-1.3.4/node/node.d/pop_stats.in munin-1.3.4-p/node/node.d/pop_stats.in
--- munin-1.3.4/node/node.d/pop_stats.in	Sat Mar  8 02:42:39 2008
+++ munin-1.3.4-p/node/node.d/pop_stats.in	Fri Apr  4 11:13:06 2008
@@ -67,7 +67,9 @@
 if ($startsize < $pos)
 {
 	# Log rotated
-	parseEximfile ($pop{'rotlogfile'}, $pos, (stat $pop{'rotlogfile'})[7]);
+	if (-f $pop{'rotlogfile'}) {
+		parseEximfile ($pop{'rotlogfile'}, $pos, (stat $pop{'rotlogfile'})[7]);
+	}
 	$pos = 0;
 }
 
diff -Naur munin-1.3.4/node/node.d/postfix_mailstats.in munin-1.3.4-p/node/node.d/postfix_mailstats.in
--- munin-1.3.4/node/node.d/postfix_mailstats.in	Sat Mar  8 02:42:39 2008
+++ munin-1.3.4-p/node/node.d/postfix_mailstats.in	Fri Apr  4 11:13:06 2008
@@ -127,7 +127,9 @@
 if ($startsize < $pos)
 {
     # Log rotated
-    parseLogfile ($rotlogfile, $pos, (stat $rotlogfile)[7]);
+    if (-f $rotlogfile) {
+        parseLogfile ($rotlogfile, $pos, (stat $rotlogfile)[7]);
+    }
     $pos = 0;
 }
 
diff -Naur munin-1.3.4/node/node.d.linux/iostat.in munin-1.3.4-p/node/node.d.linux/iostat.in
--- munin-1.3.4/node/node.d.linux/iostat.in	Sat Mar  8 02:42:39 2008
+++ munin-1.3.4-p/node/node.d.linux/iostat.in	Fri Apr  4 11:13:06 2008
@@ -146,7 +146,7 @@
 				if ($tmpnam =~ /\d+$/ ) {
 					# Special case for devices like cXdXpX,
 					# like the cciss driver
-					next unless $tmpnam =~ /\/c\d+d\d+$/
+					next unless $tmpnam =~ /\/.*d\d+$/
 				}
 				next unless grep { $_ } @fields;
 
diff -Naur munin-1.3.4/node/node.d.linux/iostat_ios.in munin-1.3.4-p/node/node.d.linux/iostat_ios.in
--- munin-1.3.4/node/node.d.linux/iostat_ios.in	Sat Mar  8 02:42:39 2008
+++ munin-1.3.4-p/node/node.d.linux/iostat_ios.in	Fri Apr  4 11:13:06 2008
@@ -37,7 +37,7 @@
     }
 }
 
-if ($ARGV[0] eq 'config') {
+if ($ARGV[0] && $ARGV[0] eq 'config') {
     print_config();
     exit;
 }
@@ -127,9 +127,12 @@
           "graph_vlabel ms\n");
 
     for my $d ( @{ get_ios(1) } ) {
-        print("$d.label $d\n",
-              "$d.type GAUGE\n",
-              "$d.draw LINE2\n");
+        print("${d}_rtime.label $d rtime\n",
+	      "${d}_wtime.label $d wtime\n",
+              "${d}_rtime.type GAUGE\n",
+              "${d}_rtime.draw LINE2\n",
+              "${d}_wtime.type GAUGE\n",
+              "${d}_wtime.draw LINE2\n");
     }
 }
 
diff -Naur munin-1.3.4/node/node.d.linux/load.in munin-1.3.4-p/node/node.d.linux/load.in
--- munin-1.3.4/node/node.d.linux/load.in	Sat Mar  8 02:42:39 2008
+++ munin-1.3.4-p/node/node.d.linux/load.in	Fri Apr  4 11:15:15 2008
@@ -35,20 +35,20 @@
 	# one machine answer for several)
 
 	# The title of the graph
-	echo 'graph_title Load average'
+	echo "graph_title Load average"
 	# Arguments to "rrdtool graph". In this case, tell it that the
 	# lower limit of the graph is '0', and that 1k=1000 (not 1024)
-	echo 'graph_args --base 1000 -l 0'
+	echo "graph_args --base 1000 -l 0"
 	# The Y-axis label
-	echo 'graph_vlabel load'
+	echo "graph_vlabel load"
 	# We want Cur/Min/Avg/Max unscaled (i.e. 0.42 load instead of
 	# 420 milliload)
-	echo 'graph_scale no'
+	echo "graph_scale no"
 	# Graph category. Defaults to 'other'
-	echo 'graph_category system'
+	echo "graph_category system"
 	# The fields. "label" is used in the legend. "label" is the only
 	# required subfield. 
-	echo 'load.label load'
+	echo "load.label load"
 	# These two are optional. They are only used if you have
 	# configured your munin to tell a Nagios-server about any
 	# problems
@@ -57,8 +57,8 @@
 	# This one is purely to add an explanation to the web page. The first
 	# one is for the graph itself, while the second one is for the field
 	# "load".
-	echo 'graph_info The load average of the machine describes how many processes are in the run-queue (scheduled to run "immediately").'
-	echo 'load.info Average load for the five minutes.'
+	echo "graph_info The load average of the machine describes how many processes are in the run-queue (scheduled to run \"immediately\")."
+	echo "load.info Average load for the five minutes."
 
 	# Last, if run with the "config"-parameter, quit here (don't
 	# display any data)
diff -Naur munin-1.3.4/node/node.d.linux/nfs_client.in munin-1.3.4-p/node/node.d.linux/nfs_client.in
--- munin-1.3.4/node/node.d.linux/nfs_client.in	Sat Mar  8 02:42:39 2008
+++ munin-1.3.4-p/node/node.d.linux/nfs_client.in	Fri Apr  4 11:13:06 2008
@@ -11,7 +11,7 @@
 proc="getattr setattr lookup access readlink read write create mkdir symlink mknod remove rmdir rename link readdir readdirplus fsstat fsinfo pathconf commit"
 
 if [ "$1" = "autoconf" ]; then
-	if [ -x "$NFS" ]; then
+	if [ -r "$NFS" ]; then
 		echo yes
 		exit 0
 	else
@@ -31,11 +31,66 @@
 	exit 0
 fi
 
-awk '/proc3/ {
-    split("'"$proc"'", names)
-    split($0,values)
-    for (e in names) {
-        printf("%s.value %d\n", names[++i], values[i+3]);
-    }
+awk '/proc2/ {
+	nfs_getattr+=$4
+	nfs_setattr+=$5
+	nfs_lookup+=$6
+	nfs_readlink+=$7
+	nfs_read+=$8
+	nfs_write+=$9
+	nfs_create+=$10
+	nfs_mkdir+=$11
+	nfs_symlink+=$12
+	nfs_remove+=$13
+	nfs_rmdir+=$14
+	nfs_rename+=$15
+	nfs_link+=$16
+	nfs_readdir+=$17
+	nfs_fsstat+=$18
+}
+/proc[34]/ {
+	nfs_getattr+=$4
+	nfs_setattr+=$5
+	nfs_lookup+=$6
+	nfs_access+=$7
+	nfs_readlink+=$8
+	nfs_read+=$9
+	nfs_write+=$10
+	nfs_create+=$11
+	nfs_mkdir+=$12
+	nfs_symlink+=$13
+	nfs_mknod+=$14
+	nfs_remove+=$15
+	nfs_rmdir+=$16
+	nfs_rename+=$17
+	nfs_link+=$18
+	nfs_readdir+=$19
+	nfs_readdirplus+=$20
+	nfs_fsstat+=$21
+	nfs_fsinfo+=$22
+	nfs_pathconf+=$23
+	nfs_commit+=$24
+}
+END {
+	print "getattr.value " nfs_getattr
+	print "setattr.value " nfs_setattr
+	print "lookup.value " nfs_lookup
+	print "access.value " nfs_access
+	print "readlink.value " nfs_readlink
+	print "read.value " nfs_read
+	print "write.value " nfs_write
+	print "create.value " nfs_create
+	print "mkdir.value " nfs_mkdir
+	print "symlink.value " nfs_symlink
+	print "mknod.value " nfs_mknod
+	print "remove.value " nfs_remove
+	print "rmdir.value " nfs_rmdir
+	print "rename.value " nfs_rename
+	print "link.value " nfs_link
+	print "readdir.value " nfs_readdir
+	print "readdirplus.value " nfs_readdirplus
+	print "fsstat.value " nfs_fsstat
+	print "fsinfo.value " nfs_fsinfo
+	print "pathconf.value " nfs_pathconf
+	print "commit.value " nfs_commit
 }' $NFS
-
diff -Naur munin-1.3.4/node/node.d.linux/nfsd.in munin-1.3.4-p/node/node.d.linux/nfsd.in
--- munin-1.3.4/node/node.d.linux/nfsd.in	Sat Mar  8 02:42:39 2008
+++ munin-1.3.4-p/node/node.d.linux/nfsd.in	Fri Apr  4 11:13:06 2008
@@ -12,7 +12,7 @@
 proc="getattr setattr lookup access readlink read write create mkdir symlink mknod remove rmdir rename link readdir readdirplus fsstat fsinfo pathconf commit"
 
 if [ "$1" = "autoconf" ]; then
-	if [ -f "$NFSD" ]; then
+	if [ -r "$NFSD" ]; then
 		echo yes
 		exit 0
 	else
@@ -32,9 +32,66 @@
 	exit 0
 fi
 
-awk '/proc3/ {
-    split("'"$proc"'", names)
-    split($0,values)
-    for (e in names)
-        printf("%s.value %d\n", names[++i], values[i+3]);
-  }' $NFSD
+awk '/proc2/ {
+	nfs_getattr+=$4
+	nfs_setattr+=$5
+	nfs_lookup+=$6
+	nfs_readlink+=$7
+	nfs_read+=$8
+	nfs_write+=$9
+	nfs_create+=$10
+	nfs_mkdir+=$11
+	nfs_symlink+=$12
+	nfs_remove+=$13
+	nfs_rmdir+=$14
+	nfs_rename+=$15
+	nfs_link+=$16
+	nfs_readdir+=$17
+	nfs_fsstat+=$18
+}
+/proc[34]/ {
+	nfs_getattr+=$4
+	nfs_setattr+=$5
+	nfs_lookup+=$6
+	nfs_access+=$7
+	nfs_readlink+=$8
+	nfs_read+=$9
+	nfs_write+=$10
+	nfs_create+=$11
+	nfs_mkdir+=$12
+	nfs_symlink+=$13
+	nfs_mknod+=$14
+	nfs_remove+=$15
+	nfs_rmdir+=$16
+	nfs_rename+=$17
+	nfs_link+=$18
+	nfs_readdir+=$19
+	nfs_readdirplus+=$20
+	nfs_fsstat+=$21
+	nfs_fsinfo+=$22
+	nfs_pathconf+=$23
+	nfs_commit+=$24
+}
+END {
+	print "getattr.value " nfs_getattr
+	print "setattr.value " nfs_setattr
+	print "lookup.value " nfs_lookup
+	print "access.value " nfs_access
+	print "readlink.value " nfs_readlink
+	print "read.value " nfs_read
+	print "write.value " nfs_write
+	print "create.value " nfs_create
+	print "mkdir.value " nfs_mkdir
+	print "symlink.value " nfs_symlink
+	print "mknod.value " nfs_mknod
+	print "remove.value " nfs_remove
+	print "rmdir.value " nfs_rmdir
+	print "rename.value " nfs_rename
+	print "link.value " nfs_link
+	print "readdir.value " nfs_readdir
+	print "readdirplus.value " nfs_readdirplus
+	print "fsstat.value " nfs_fsstat
+	print "fsinfo.value " nfs_fsinfo
+	print "pathconf.value " nfs_pathconf
+	print "commit.value " nfs_commit
+}' $NFSD
--- munin-1.3.4/node/node.d.linux/ip_.in~	2008-03-08 02:42:39.000000000 +0100
+++ munin-1.3.4/node/node.d.linux/ip_.in	2008-05-20 13:00:17.000000000 +0200
@@ -40,16 +40,16 @@
 	# This is a fun hack to make the plugin ip6 compatible.
 	# Suggested in ticket #439 by "jodal".
 	eval 'function iptables() {
-	    /sbin/ip6tables "$@"
+	    /usr/sbin/ip6tables "$@"
 	}'
 	;;
 esac
 
 if [ "$1" = "autoconf" ]; then
 	if [ -r /proc/net/dev ]; then
-		iptables -L ${INPUT} -v -n -x >/dev/null 2>/dev/null
+		/usr/sbin/iptables -L ${INPUT} -v -n -x >/dev/null 2>/dev/null
 		if [ $? -gt 0 ]; then
-			echo "no (could not run iptables as user `whoami`)"
+			echo "no (could not run /usr/sbin/iptables as user `whoami`)"
 			exit 1
 		else
 			echo yes
@@ -62,7 +62,7 @@
 fi
 
 if [ "$1" = "suggest" ]; then
-	iptables -L ${INPUT} -v -n -x 2>/dev/null | awk '$8 ~ /[0-9]/ { if (done[$8]!=1) {print $8; done[$8]=1;}}'
+	/usr/sbin/iptables -L ${INPUT} -v -n -x 2>/dev/null | awk '$8 ~ /[0-9]/ { if (done[$8]!=1) {print $8; done[$8]=1;}}'
 	exit 0
 fi
 
@@ -84,6 +84,6 @@
         exit 0
 fi;
 
-iptables -L ${INPUT} -v -n -x | grep -m1 $IP | awk "{ print \"in.value \" \$2 }"
-iptables -L ${OUTPUT} -v -n -x | grep -m1 $IP | awk "{ print \"out.value \" \$2 }"
+/usr/sbin/iptables -L ${INPUT} -v -n -x | grep -m1 $IP | awk "{ print \"in.value \" \$2 }"
+/usr/sbin/iptables -L ${OUTPUT} -v -n -x | grep -m1 $IP | awk "{ print \"out.value \" \$2 }"
 
--- munin-1.3.4/node/node.d.linux/vlan_inetuse_.in~	2008-03-08 02:42:39.000000000 +0100
+++ munin-1.3.4/node/node.d.linux/vlan_inetuse_.in	2008-05-20 13:00:50.000000000 +0200
@@ -37,8 +37,8 @@
 my %in_octets = ();
 my %out_octets = ();
 
-open (IN, "/sbin/iptables -v -x -L $INTERFACE-in |") or 
-	die "Could not run iptables: $!\n";
+open (IN, "/usr/sbin/iptables -v -x -L $INTERFACE-in |") or 
+	die "Could not run /usr/sbin/iptables: $!\n";
 while (<IN>)
 {
 	if (/^\s*\d+\s+(\d+)  +([a-z]+)\s+\S+\s+\S+\s+\S+\s+\S+\s+\S+(?:\s+|)(.+|)$/)
@@ -49,10 +49,10 @@
 	}
 }
 close IN;
-die "Error running iptables. Dying\n" if $?;
+die "Error running /usr/sbin/iptables. Dying\n" if $?;
 
-open (IN, "/sbin/iptables -v -x -L $INTERFACE-out |") or 
-	die "Could not run iptables: $!\n";
+open (IN, "/usr/sbin/iptables -v -x -L $INTERFACE-out |") or 
+	die "Could not run /usr/sbin/iptables: $!\n";
 while (<IN>)
 {
 	if (/^\s*\d+\s+(\d+)  +([a-z]+)\s+\S+\s+\S+\s+\S+\s+\S+\s+\S+(?:\s+|)(.+|)$/)
@@ -67,7 +67,7 @@
 	}
 }
 close IN;
-die "Error running iptables. Dying\n" if $?;
+die "Error running /usr/sbin/iptables. Dying\n" if $?;
 
 if ($ARGV[0] and $ARGV[0] eq "config")
 {
--- munin-1.3.4/node/node.d.linux/vlan_linkuse_.in~	2008-03-08 02:42:39.000000000 +0100
+++ munin-1.3.4/node/node.d.linux/vlan_linkuse_.in	2008-05-20 13:01:48.000000000 +0200
@@ -29,8 +29,8 @@
 my %in_octets = ();
 my %out_octets = ();
 
-open (IN, "/sbin/iptables -v -x -L $INTERFACE-in |") or 
-	die "Could not run iptables: $!\n";
+open (IN, "/usr/sbin/iptables -v -x -L $INTERFACE-in |") or 
+	die "Could not run /usr/sbin/iptables: $!\n";
 while (<IN>)
 {
 	if (/^\s*\d+\s+(\d+)  +([a-z]+)\s+\S+\s+\S+\s+\S+\s+\S+\s+\S+(?:\s+|)(.+|)$/)
@@ -58,10 +58,10 @@
 	}
 }
 close IN;
-die "Error running iptables. Dying\n" if $?;
+die "Error running /usr/sbin/iptables. Dying\n" if $?;
 
-open (IN, "/sbin/iptables -v -x -L $INTERFACE-out |") or 
-	die "Could not run iptables: $!\n";
+open (IN, "/usr/sbin/iptables -v -x -L $INTERFACE-out |") or 
+	die "Could not run /usr/sbin/iptables: $!\n";
 while (<IN>)
 {
 	if (/^\s*\d+\s+(\d+)  +([a-z]+)\s+\S+\s+\S+\s+\S+\s+\S+\s+\S+(?:\s+|)(.+|)$/)
@@ -84,5 +84,5 @@
 	}
 }
 close IN;
-die "Error running iptables. Dying\n" if $?;
+die "Error running /usr/sbin/iptables. Dying\n" if $?;
 # vim:syntax=perl
