--- munin-1.3.4/node/munin-run.in	2009-01-26 11:40:43.000000000 +0100
+++ munin-1.3.4/node/munin-run.in.new	2009-01-26 11:40:26.000000000 +0100
@@ -248,7 +248,7 @@
 sub load_services {
     if ($sconffile)
     {
-	if (!&load_auth_file ("", $sconffile, \%sconf))
+	if (!&load_auth_file ("", $sconffile, \%sconf, $ARGV[0]))
 	{
 	    warn "Something wicked happened while reading \"$sconffile\". Check the previous log lines for spesifics.";
 	}
@@ -267,7 +267,7 @@
 		{
 			next FILES if $file =~ /$regex/;
 		}
-		if (!&load_auth_file ($sconfdir, $file, \%sconf))
+		if (!&load_auth_file ($sconfdir, $file, \%sconf, $ARGV[0]))
 		{
 		    warn "Something wicked happened while reading \"$servicedir/$file\". Check the previous log lines for spesifics.";
 		}
@@ -467,7 +467,7 @@
 
 sub load_auth_file 
 {
-    my ($dir, $file, $sconf) = @_;
+    my ($dir, $file, $sconf, $plug) = @_;
     my $service = $file;
 
     if (!defined $dir or !defined $file or !defined $sconf)
@@ -496,7 +496,7 @@
 	{
 	    my $tmpid = $1;
 	    $sconf->{$service}{'user'} = &get_uid ($tmpid);
-	    if (!defined $sconf->{$service}{'user'})
+	    if (!defined $sconf->{$service}{'user'} and $plug =~ $service)
 	    {
 		warn "User \"$tmpid\" in configuration file \"$dir/$file\" nonexistant. Skipping plugin.";
 		next;
@@ -516,7 +516,7 @@
 		}
 
 		my $g = &get_gid ($group);
-		if (!defined $g and !$optional)
+		if (!defined $g and !$optional and $plug =~ $service)
 		{
 		    warn "Group \"$group\" in configuration file \"$dir/$file\" nonexistant. Skipping plugin.";
 		    next;
@@ -574,7 +574,7 @@
             print "# Saving $service->env->lrrd_$1 = $2...\n" if $DEBUG;
 	    warn "Warning: Deprecated format in \"$dir/$file\" under \"[$service]\" (\"$1 $2\" should be rewritten to \"env.$1 $2\").";
         } 
-        elsif (/\S/)
+        elsif (/\S/ and $plug =~ $service)
         {
             warn "Warning: Unknown config option in \"$dir/$file\" under \"[$service]\": $_";
         }
