diff -ur munin-1.3.2/dists/tarball/plugins.conf munin-1.3.2-plugins/dists/tarball/plugins.conf
--- munin-1.3.2/dists/tarball/plugins.conf	2005-04-15 22:29:18.000000000 +0200
+++ munin-1.3.2-plugins/dists/tarball/plugins.conf	2005-10-09 14:06:36.770701544 +0200
@@ -29,3 +29,64 @@
 
 [postfix*]
 user root
+env.logdir /var/log
+env.logfile maillog
+
+[netstat]
+group proc
+
+[amavis]
+user root
+env.logfile /var/log/maillog
+
+[courier*]
+user root
+env.logfile /var/log/maillog
+
+[mailman]
+user mailman
+group mailman
+
+[bind9_rndc]
+group named
+env.querystats /var/lib/named/named.stats
+
+[squid*]
+group squid
+#env.squidport 3128
+#env.squiduser manager
+#env.squidpasswd <SECRET>
+
+[fw*]
+group proc
+
+[if*]
+group proc
+
+[dhcpd3]
+user root
+
+[ip_*]
+group proc
+
+[munin*]
+user munin
+
+[ping_*]
+group adm
+
+[port_*]
+group proc
+
+[processes]
+group proc
+
+[ps_*]
+group proc
+
+[psu_*]
+group proc
+
+[nfs*]
+group proc
+
diff -ur munin-1.3.2/node/node.d/bind9.in munin-1.3.2-plugins/node/node.d/bind9.in
--- munin-1.3.2/node/node.d/bind9.in	2005-04-15 22:29:18.000000000 +0200
+++ munin-1.3.2-plugins/node/node.d/bind9.in	2005-10-09 13:56:09.446270008 +0200
@@ -19,7 +19,7 @@
 my %IN;
 
 sub get_state {
-    open(Q,"< $STATEFILE") or die;
+    open(Q,"< $STATEFILE") or return;
     while (<Q>) {
         chomp;
         my ($q,$n) = split(/\s+/,$_,2);
diff -ur munin-1.3.2/node/node.d/courier_.in munin-1.3.2-plugins/node/node.d/courier_.in
--- munin-1.3.2/node/node.d/courier_.in	2005-04-15 22:29:18.000000000 +0200
+++ munin-1.3.2-plugins/node/node.d/courier_.in	2005-10-09 13:56:34.916397960 +0200
@@ -34,9 +34,9 @@
 # Set the location of the courier logs
 COURIER_LOG=${logfile:-/var/log/mail.log}
 SERVICE=${service:-`basename $0 | sed 's/^courier_//g'`}
-TEMP_FILE=`mktemp -p /tmp/ munin-courier.XXXXXX`
+TEMP_FILE=`mktemp /tmp/munin-courier.XXXXXX`
 OFFSET_FILE=@@PLUGSTATE@@/courier_${SERVICE}.offset
-LOGTAIL=${logtail:-/usr/sbin/logtail}
+LOGTAIL=${logtail:-`which logtail`}
 
 if [ ! -f "$TEMP_FILE" ]; then
     exit 3
diff -ur munin-1.3.2/node/node.d/courier_mta_mailstats.in munin-1.3.2-rotlog/node/node.d/courier_mta_mailstats.in
--- munin-1.3.2/node/node.d/courier_mta_mailstats.in	2005-04-15 22:29:18.000000000 +0200
+++ munin-1.3.2-rotlog/node/node.d/courier_mta_mailstats.in	2005-10-10 22:18:45.170285184 +0200
@@ -101,7 +101,9 @@
 
 if($startsize < $pos) {
     # Log rotated
-    parseLogfile ($rotlogfile, $pos, (stat $rotlogfile)[7]);
+    if (-f $rotlogfile) {
+        parseLogfile ($rotlogfile, $pos, (stat $rotlogfile)[7]);
+    }
     $pos = 0;
 }
 
diff -ur munin-1.3.2/node/node.d/courier_mta_mailvolume.in munin-1.3.2-rotlog/node/node.d/courier_mta_mailvolume.in
--- munin-1.3.2/node/node.d/courier_mta_mailvolume.in	2005-04-15 22:29:18.000000000 +0200
+++ munin-1.3.2-rotlog/node/node.d/courier_mta_mailvolume.in	2005-10-10 22:19:00.722920824 +0200
@@ -92,7 +92,9 @@
 
 if($startsize < $pos) {
     # Log rotated
-    parseLogfile ($rotlogfile, $pos, (stat $rotlogfile)[7]);
+    if (-f $rotlogfile) {
+        parseLogfile ($rotlogfile, $pos, (stat $rotlogfile)[7]);
+    }
     $pos = 0;
 }
 
diff -ur munin-1.3.2/node/node.d/cupsys_pages.in munin-1.3.2-rotlog/node/node.d/cupsys_pages.in
--- munin-1.3.2/node/node.d/cupsys_pages.in	2005-04-15 22:29:18.000000000 +0200
+++ munin-1.3.2-rotlog/node/node.d/cupsys_pages.in	2005-10-10 22:19:22.973538216 +0200
@@ -91,7 +91,9 @@
 }
 if ($startsize < $pos) {
     # Log rotated
-    parselogfile ($rotlogfile, $pos, (stat $rotlogfile)[7]);
+    if (-f $rotlogfile) {
+        parselogfile ($rotlogfile, $pos, (stat $rotlogfile)[7]);
+    }
     $pos = 0;
 }
 
diff -ur munin-1.3.2/node/node.d/exim_mailstats.in munin-1.3.2-rotlog/node/node.d/exim_mailstats.in
--- munin-1.3.2/node/node.d/exim_mailstats.in	2005-04-15 22:29:18.000000000 +0200
+++ munin-1.3.2-rotlog/node/node.d/exim_mailstats.in	2005-10-10 22:19:45.239153328 +0200
@@ -183,7 +183,9 @@
 if ($startsize < $pos)
 {
     # Log rotated
-    parseEximfile ($rotlogfile, $pos, (stat $rotlogfile)[7]);
+    if (-f $rotlogfile) {
+        parseEximfile ($rotlogfile, $pos, (stat $rotlogfile)[7]);
+    }
     $pos = 0;
 }
 
diff -ur munin-1.3.2/node/node.d/mailman.in munin-1.3.2-plugins/node/node.d/mailman.in
--- munin-1.3.2/node/node.d/mailman.in	2005-04-15 22:29:18.000000000 +0200
+++ munin-1.3.2-plugins/node/node.d/mailman.in	2005-10-09 14:04:27.503353168 +0200
@@ -67,7 +67,9 @@
 
 if($startsize < $pos) {
     # Log rotated
-    parseMailmanLog($rotlogfile, $pos, (stat $rotlogfile)[7]);
+    if (-f $rotlogfile) {
+        parseMailmanLog($rotlogfile, $pos, (stat $rotlogfile)[7]);
+    }
     $pos = 0;
 }
 
diff -ur munin-1.3.2/node/node.d.linux/fw_forwarded_local.in munin-1.3.2-plugins/node/node.d.linux/fw_forwarded_local.in
--- munin-1.3.2/node/node.d.linux/fw_forwarded_local.in	2005-04-15 22:29:18.000000000 +0200
+++ munin-1.3.2-plugins/node/node.d.linux/fw_forwarded_local.in	2005-10-09 14:02:02.324423720 +0200
@@ -39,10 +39,8 @@
 	echo 'graph_category network'
 	echo 'forward.label forward'
 	echo 'forward.type GAUGE'
-	echo 'forward.max 500'''
 	echo 'local.label local'
 	echo 'local.type GAUGE'
-	echo 'local.max 500'''
 	exit 0
 fi
 
diff -ur munin-1.3.2/node/node.d.linux/iostat_ios.in munin-1.3.2-plugins/node/node.d.linux/iostat_ios.in
--- munin-1.3.2/node/node.d.linux/iostat_ios.in	2005-04-15 22:29:18.000000000 +0200
+++ munin-1.3.2-plugins/node/node.d.linux/iostat_ios.in	2005-10-09 14:03:10.930993936 +0200
@@ -27,7 +27,7 @@
 use constant STATEFILE => '@@PLUGSTATE@@/iostat-ios.state';
 
 
-if ($ARGV[0] eq 'config') {
+if ($ARGV[0] && $ARGV[0] eq 'config') {
     print_config();
     exit;
 }
@@ -52,7 +52,7 @@
 
 
 sub filter {
-    my ($major, $minor, $tmpnam);
+    my ($major, $minor, $tmpnam) = @_;
     return 0 if ($major ==   1); # RAM devices
     return 0 if ($major ==   9); # MD devices
     return 0 if ($major ==  58); # LVM devices
@@ -122,9 +122,12 @@
           "graph_vlabel ms\n");
 
     for my $d ( @{ get_ios(1) } ) {
-        print("$d.label $d\n",
-              "$d.type GAUGE\n",
-              "$d.draw LINE2\n");
+        print("${d}_rtime.label $d rtime\n",
+	      "${d}_wtime.label $d wtime\n",
+              "${d}_rtime.type GAUGE\n",
+              "${d}_rtime.draw LINE2\n",
+              "${d}_wtime.type GAUGE\n",
+              "${d}_wtime.draw LINE2\n");
     }
 }
 
diff -ur munin-1.3.2/node/node.d.linux/nfs_client.in munin-1.3.2-plugins/node/node.d.linux/nfs_client.in
--- munin-1.3.2/node/node.d.linux/nfs_client.in	2005-04-15 22:29:18.000000000 +0200
+++ munin-1.3.2-plugins/node/node.d.linux/nfs_client.in	2005-10-09 14:04:42.692044136 +0200
@@ -10,7 +10,7 @@
 proc="getattr setattr lookup access readlink read write create mkdir symlink mknod remove rmdir rename link readdir readdirplus fsstat fsinfo pathconf commit"
 
 if [ "$1" = "autoconf" ]; then
-	if [ -x "$NFS" ]; then
+	if [ -r "$NFS" ]; then
 		echo yes
 		exit 0
 	else
@@ -30,12 +30,67 @@
 	exit 0
 fi
 
-i=4;
-
-for a in $proc; do
-	echo -n "$a.value "
-	grep proc3 $NFS \
-		| cut -f $i -d ' ' \
-		| awk '{print $1}'
-	i=$(expr $i + 1)
-done
+cat $NFS | \
+awk '/proc2/ {
+	nfs_getattr+=$4
+	nfs_setattr+=$5
+	nfs_lookup+=$6
+	nfs_readlink+=$7
+	nfs_read+=$8
+	nfs_write+=$9
+	nfs_create+=$10
+	nfs_mkdir+=$11
+	nfs_symlink+=$12
+	nfs_remove+=$13
+	nfs_rmdir+=$14
+	nfs_rename+=$15
+	nfs_link+=$16
+	nfs_readdir+=$17
+	nfs_fsstat+=$18
+}
+/proc[34]/ {
+	nfs_getattr+=$4
+	nfs_setattr+=$5
+	nfs_lookup+=$6
+	nfs_access+=$7
+	nfs_readlink+=$8
+	nfs_read+=$9
+	nfs_write+=$10
+	nfs_create+=$11
+	nfs_mkdir+=$12
+	nfs_symlink+=$13
+	nfs_mknod+=$14
+	nfs_remove+=$15
+	nfs_rmdir+=$16
+	nfs_rename+=$17
+	nfs_link+=$18
+	nfs_readdir+=$19
+	nfs_readdirplus+=$20
+	nfs_fsstat+=$21
+	nfs_fsinfo+=$22
+	nfs_pathconf+=$23
+	nfs_commit+=$24
+}
+END {
+	print "getattr.value " nfs_getattr
+	print "setattr.value " nfs_setattr
+	print "lookup.value " nfs_lookup
+	print "access.value " nfs_access
+	print "readlink.value " nfs_readlink
+	print "read.value " nfs_read
+	print "write.value " nfs_write
+	print "create.value " nfs_create
+	print "mkdir.value " nfs_mkdir
+	print "symlink.value " nfs_symlink
+	print "mknod.value " nfs_mknod
+	print "remove.value " nfs_remove
+	print "rmdir.value " nfs_rmdir
+	print "rename.value " nfs_rename
+	print "link.value " nfs_link
+	print "readdir.value " nfs_readdir
+	print "readdirplus.value " nfs_readdirplus
+	print "fsstat.value " nfs_fsstat
+	print "fsinfo.value " nfs_fsinfo
+	print "pathconf.value " nfs_pathconf
+	print "commit.value " nfs_commit
+}'
diff -ur munin-1.3.2/node/node.d.linux/nfsd.in munin-1.3.2-plugins/node/node.d.linux/nfsd.in
--- munin-1.3.2/node/node.d.linux/nfsd.in	2005-04-15 22:29:18.000000000 +0200
+++ munin-1.3.2-plugins/node/node.d.linux/nfsd.in	2005-10-09 14:04:49.220051728 +0200
@@ -10,7 +10,7 @@
 proc="getattr setattr lookup access readlink read write create mkdir symlink mknod remove rmdir rename link readdir readdirplus fsstat fsinfo pathconf commit"
 
 if [ "$1" = "autoconf" ]; then
-	if [ -x "$NFSD" ]; then
+	if [ -r "$NFSD" ]; then
 		echo yes
 		exit 0
 	else
@@ -30,12 +30,67 @@
 	exit 0
 fi
 
-i=4;
-
-for a in $proc; do
-	echo -n "$a.value "
-	grep proc3 $NFSD \
-		| cut -f $i -d ' ' \
-		| awk '{print $1}'
-	i=$(expr $i + 1)
-done
+cat $NFSD | \
+awk '/proc2/ {
+	nfs_getattr+=$4
+	nfs_setattr+=$5
+	nfs_lookup+=$6
+	nfs_readlink+=$7
+	nfs_read+=$8
+	nfs_write+=$9
+	nfs_create+=$10
+	nfs_mkdir+=$11
+	nfs_symlink+=$12
+	nfs_remove+=$13
+	nfs_rmdir+=$14
+	nfs_rename+=$15
+	nfs_link+=$16
+	nfs_readdir+=$17
+	nfs_fsstat+=$18
+}
+/proc[34]/ {
+	nfs_getattr+=$4
+	nfs_setattr+=$5
+	nfs_lookup+=$6
+	nfs_access+=$7
+	nfs_readlink+=$8
+	nfs_read+=$9
+	nfs_write+=$10
+	nfs_create+=$11
+	nfs_mkdir+=$12
+	nfs_symlink+=$13
+	nfs_mknod+=$14
+	nfs_remove+=$15
+	nfs_rmdir+=$16
+	nfs_rename+=$17
+	nfs_link+=$18
+	nfs_readdir+=$19
+	nfs_readdirplus+=$20
+	nfs_fsstat+=$21
+	nfs_fsinfo+=$22
+	nfs_pathconf+=$23
+	nfs_commit+=$24
+}
+END {
+	print "getattr.value " nfs_getattr
+	print "setattr.value " nfs_setattr
+	print "lookup.value " nfs_lookup
+	print "access.value " nfs_access
+	print "readlink.value " nfs_readlink
+	print "read.value " nfs_read
+	print "write.value " nfs_write
+	print "create.value " nfs_create
+	print "mkdir.value " nfs_mkdir
+	print "symlink.value " nfs_symlink
+	print "mknod.value " nfs_mknod
+	print "remove.value " nfs_remove
+	print "rmdir.value " nfs_rmdir
+	print "rename.value " nfs_rename
+	print "link.value " nfs_link
+	print "readdir.value " nfs_readdir
+	print "readdirplus.value " nfs_readdirplus
+	print "fsstat.value " nfs_fsstat
+	print "fsinfo.value " nfs_fsinfo
+	print "pathconf.value " nfs_pathconf
+	print "commit.value " nfs_commit
+}'
diff -ur munin-1.3.2/node/node.d/perdition.in munin-1.3.2-plugins/node/node.d/perdition.in
--- munin-1.3.2/node/node.d/perdition.in	2005-04-15 22:29:18.000000000 +0200
+++ munin-1.3.2-plugins/node/node.d/perdition.in	2005-10-09 18:50:05.680496376 +0200
@@ -26,9 +26,9 @@
 
 # Set the location of the perdition logs
 PERDITION_LOG=${logfile:-/var/log/perdition.log}
-TEMP_FILE=`mktemp -p /tmp/ munin-perdition.XXXXX`
+TEMP_FILE=`mktemp /tmp/munin-perdition.XXXXX`
 OFFSET_FILE=@@PLUGSTATE@@/perdition.offset
-LOGTAIL=${logtail:-/usr/sbin/logtail}
+LOGTAIL=${logtail:-`which logtail`}
 
 if [ ! -f "$TEMP_FILE" ]; then
     exit 3
diff -ur munin-1.3.2/node/node.d/pop_stats.in munin-1.3.2-rotlog/node/node.d/pop_stats.in
--- munin-1.3.2/node/node.d/pop_stats.in	2005-04-15 22:29:18.000000000 +0200
+++ munin-1.3.2-rotlog/node/node.d/pop_stats.in	2005-10-10 22:20:53.958706368 +0200
@@ -67,7 +67,9 @@
 if ($startsize < $pos)
 {
 	# Log rotated
-	parseEximfile ($pop{'rotlogfile'}, $pos, (stat $pop{'rotlogfile'})[7]);
+	if (-f $pop{'rotlogfile'}) {
+		parseEximfile ($pop{'rotlogfile'}, $pos, (stat $pop{'rotlogfile'})[7]);
+	}
 	$pos = 0;
 }
 
diff -ur munin-1.3.2/node/node.d/postfix_mailstats.in munin-1.3.2-rotlog/node/node.d/postfix_mailstats.in
--- munin-1.3.2/node/node.d/postfix_mailstats.in	2005-04-15 22:29:18.000000000 +0200
+++ munin-1.3.2-rotlog/node/node.d/postfix_mailstats.in	2005-10-10 22:21:09.989269352 +0200
@@ -122,7 +122,9 @@
 if ($startsize < $pos)
 {
     # Log rotated
-    parseLogfile ($rotlogfile, $pos, (stat $rotlogfile)[7]);
+    if (-f $rotlogfile) {
+        parseLogfile ($rotlogfile, $pos, (stat $rotlogfile)[7]);
+    }
     $pos = 0;
 }
 
diff -ur munin-1.3.2/node/node.d/postfix_mailvolume.in munin-1.3.2-rotlog/node/node.d/postfix_mailvolume.in
--- munin-1.3.2/node/node.d/postfix_mailvolume.in	2005-04-15 22:29:18.000000000 +0200
+++ munin-1.3.2-rotlog/node/node.d/postfix_mailvolume.in	2005-10-10 22:21:17.923063232 +0200
@@ -111,7 +111,9 @@
 if ($startsize < $pos)
 {
     # Log rotated
-    parseLogfile ($rotlogfile, $pos, (stat $rotlogfile)[7]);
+    if (-f $rotlogfile) {
+        parseLogfile ($rotlogfile, $pos, (stat $rotlogfile)[7]);
+    }
     $pos = 0;
 }
 
diff -ur munin-1.3.2/node/node.d/spamstats.in munin-1.3.2-rotlog/node/node.d/spamstats.in
--- munin-1.3.2/node/node.d/spamstats.in	2005-04-15 22:29:18.000000000 +0200
+++ munin-1.3.2-rotlog/node/node.d/spamstats.in	2005-10-10 22:21:26.427770320 +0200
@@ -72,7 +72,9 @@
 if ($startsize < $pos)
 {
     # Log rotated
-    parselogfile ($rotlogfile, $pos, (stat $rotlogfile)[7]);
+    if (-f $rotlogfile) {
+        parselogfile ($rotlogfile, $pos, (stat $rotlogfile)[7]);
+    }
     $pos = 0;
 }
 
diff -ur munin-1.3.2/node/node.d/apache_accesses.in munin-1.3.2-apache/node/node.d/apache_accesses.in
--- munin-1.3.2/node/node.d/apache_accesses.in	2005-04-15 22:29:18.000000000 +0200
+++ munin-1.3.2-apache/node/node.d/apache_accesses.in	2005-10-10 22:45:35.341501952 +0200
@@ -36,6 +36,7 @@
 
 my $URL = exists $ENV{'url'} ? $ENV{'url'} : "http://127.0.0.1:%d/server-status?auto";
 my @PORTS = exists $ENV{'ports'} ? split(' ', $ENV{'ports'}) : (80);
+my $HOST = exists $ENV{'host'} ? $ENV{'host'} : undef;
 
 if ( exists $ARGV[0] and $ARGV[0] eq "autoconf" )
 {
@@ -64,7 +65,11 @@
 
 if ( exists $ARGV[0] and $ARGV[0] eq "config" )
 {
-	print "graph_title Apache accesses\n";
+	print "graph_title Apache accesses";
+	if ( $HOST ) {
+		print " for $HOST";
+	}
+	print "\n";
 	print "graph_args --base 1000\n";
 	print "graph_vlabel accesses / \${graph_period}\n";
 	print "graph_category apache\n";
diff -ur munin-1.3.2/node/node.d/apache_processes.in munin-1.3.2-apache/node/node.d/apache_processes.in
--- munin-1.3.2/node/node.d/apache_processes.in	2005-04-15 22:29:18.000000000 +0200
+++ munin-1.3.2-apache/node/node.d/apache_processes.in	2005-10-10 22:45:01.013720568 +0200
@@ -34,6 +34,7 @@
 
 my $URL = exists $ENV{'url'} ? $ENV{'url'} : "http://127.0.0.1:%d/server-status?auto";
 my @PORTS = exists $ENV{'ports'} ? split(' ', $ENV{'ports'}) : (80);
+my $HOST = exists $ENV{'host'} ? $ENV{'host'} : undef;
 
 if ( exists $ARGV[0] and $ARGV[0] eq "autoconf" )
 {
@@ -65,7 +66,11 @@
 
 if ( exists $ARGV[0] and $ARGV[0] eq "config" )
 {
-        print "graph_title Apache processes\n";
+        print "graph_title Apache processes";
+	if ( $HOST ) {
+		print " for $HOST";
+	}
+	print "\n";
         print "graph_args --base 1000 -l 0\n";
 		print "graph_category apache\n";
 	print "graph_order ";
diff -ur munin-1.3.2/node/node.d/apache_volume.in munin-1.3.2-apache/node/node.d/apache_volume.in
--- munin-1.3.2/node/node.d/apache_volume.in	2005-04-15 22:29:18.000000000 +0200
+++ munin-1.3.2-apache/node/node.d/apache_volume.in	2005-10-10 22:45:30.091300104 +0200
@@ -35,6 +35,7 @@
 
 my $URL = exists $ENV{'url'} ? $ENV{'url'} : "http://127.0.0.1:%d/server-status?auto";
 my @PORTS = exists $ENV{'ports'} ? split(' ', $ENV{'ports'}) : (80);
+my $HOST = exists $ENV{'host'} ? $ENV{'host'} : undef;
 
 if ( exists $ARGV[0] and $ARGV[0] eq "autoconf" )
 {
@@ -63,7 +64,11 @@
 
 if ( exists $ARGV[0] and $ARGV[0] eq "config" )
 {
-	print "graph_title Apache volume\n";
+	print "graph_title Apache volume";
+	if ( $HOST ) {
+		print " for $HOST";
+	}
+	print "\n";
 	print "graph_args --base 1000\n";
 	print "graph_vlabel bytes per \${graph_period}\n";
 	print "graph_category apache\n";
diff -ur munin-1.3.2/node/node.d/mysql_slowqueries.in munin-1.3.2-mysql/node/node.d/mysql_slowqueries.in
--- munin-1.3.2/node/node.d/mysql_slowqueries.in	2005-04-15 22:29:18.000000000 +0200
+++ munin-1.3.2-mysql/node/node.d/mysql_slowqueries.in	2005-10-11 16:12:34.267293304 +0200
@@ -47,5 +47,5 @@
 	exit 0
 fi
 
-/usr/bin/printf "queries.value "
+/bin/printf "queries.value "
 ($MYSQLADMIN $MYSQLOPTS status 2>/dev/null || echo a a a a a a a a U) | awk '{print $9}'
diff -ur munin-1.3.2/node/node.d/mysql_threads.in munin-1.3.2-mysql/node/node.d/mysql_threads.in
--- munin-1.3.2/node/node.d/mysql_threads.in	2005-04-15 22:29:18.000000000 +0200
+++ munin-1.3.2-mysql/node/node.d/mysql_threads.in	2005-10-11 16:12:29.286050568 +0200
@@ -45,5 +45,5 @@
 	exit 0
 fi
 
-/usr/bin/printf "threads.value "
+/bin/printf "threads.value "
 ($MYSQLADMIN $MYSQLOPTS status 2>/dev/null || echo 'a a a U') | awk '{print $4}'
